// @radix-ui/colors/index.js
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

const grayDark = {
    gray1: "#111111",
    gray2: "#191919",
    gray3: "#222222",
    gray4: "#2a2a2a",
    gray5: "#313131",
    gray6: "#3a3a3a",
    gray7: "#484848",
    gray8: "#606060",
    gray9: "#6e6e6e",
    gray10: "#7b7b7b",
    gray11: "#b3b3b3",
    gray12: "#eeeeee",
};
const grayDarkA = {
    grayA1: "#00000000",
    grayA2: "#ffffff09",
    grayA3: "#ffffff12",
    grayA4: "#ffffff1b",
    grayA5: "#ffffff22",
    grayA6: "#ffffff2c",
    grayA7: "#ffffff3b",
    grayA8: "#ffffff55",
    grayA9: "#ffffff64",
    grayA10: "#ffffff72",
    grayA11: "#ffffffae",
    grayA12: "#ffffffed",
};
const grayDarkP3 = {
    gray1: "color(display-p3 0.0667 0.0667 0.0667)",
    gray2: "color(display-p3 0.098 0.098 0.098)",
    gray3: "color(display-p3 0.1348 0.1348 0.1348)",
    gray4: "color(display-p3 0.1632 0.1632 0.1632)",
    gray5: "color(display-p3 0.1916 0.1916 0.1916)",
    gray6: "color(display-p3 0.2282 0.2282 0.2282)",
    gray7: "color(display-p3 0.2834 0.2834 0.2834)",
    gray8: "color(display-p3 0.375 0.375 0.375)",
    gray9: "color(display-p3 0.4314 0.4314 0.4314)",
    gray10: "color(display-p3 0.4832 0.4832 0.4832)",
    gray11: "color(display-p3 0.7 0.7 0.7)",
    gray12: "color(display-p3 0.9333 0.9333 0.9333)",
};
const grayDarkP3A = {
    grayA1: "color(display-p3 0 0 0 / 0)",
    grayA2: "color(display-p3 1 1 1 / 0.0336)",
    grayA3: "color(display-p3 1 1 1 / 0.0714)",
    grayA4: "color(display-p3 1 1 1 / 0.105)",
    grayA5: "color(display-p3 1 1 1 / 0.1345)",
    grayA6: "color(display-p3 1 1 1 / 0.1723)",
    grayA7: "color(display-p3 1 1 1 / 0.2311)",
    grayA8: "color(display-p3 1 1 1 / 0.3319)",
    grayA9: "color(display-p3 1 1 1 / 0.3908)",
    grayA10: "color(display-p3 1 1 1 / 0.4454)",
    grayA11: "color(display-p3 1 1 1 / 0.6807)",
    grayA12: "color(display-p3 1 1 1 / 0.9286)",
};
const mauveDark = {
    mauve1: "#121113",
    mauve2: "#1a191b",
    mauve3: "#232225",
    mauve4: "#2b292d",
    mauve5: "#323035",
    mauve6: "#3c393f",
    mauve7: "#49474e",
    mauve8: "#625f69",
    mauve9: "#6f6d78",
    mauve10: "#7c7a85",
    mauve11: "#b5b2bc",
    mauve12: "#eeeef0",
};
const mauveDarkA = {
    mauveA1: "#00000000",
    mauveA2: "#f5f4f609",
    mauveA3: "#ebeaf814",
    mauveA4: "#eee5f81d",
    mauveA5: "#efe6fe25",
    mauveA6: "#f1e6fd30",
    mauveA7: "#eee9ff40",
    mauveA8: "#eee7ff5d",
    mauveA9: "#eae6fd6e",
    mauveA10: "#ece9fd7c",
    mauveA11: "#f5f1ffb7",
    mauveA12: "#fdfdffef",
};
const mauveDarkP3 = {
    mauve1: "color(display-p3 0.0699 0.0668 0.0739)",
    mauve2: "color(display-p3 0.1013 0.0982 0.1053)",
    mauve3: "color(display-p3 0.1382 0.1338 0.1444)",
    mauve4: "color(display-p3 0.1667 0.1614 0.1747)",
    mauve5: "color(display-p3 0.1957 0.1895 0.2057)",
    mauve6: "color(display-p3 0.232 0.2247 0.2446)",
    mauve7: "color(display-p3 0.2856 0.277 0.3024)",
    mauve8: "color(display-p3 0.3832 0.3728 0.4085)",
    mauve9: "color(display-p3 0.4339 0.4277 0.4671)",
    mauve10: "color(display-p3 0.4867 0.4801 0.5189)",
    mauve11: "color(display-p3 0.7074 0.6997 0.7354)",
    mauve12: "color(display-p3 0.9333 0.9333 0.9405)",
};
const mauveDarkP3A = {
    mauveA1: "color(display-p3 0 0 0 / 0)",
    mauveA2: "color(display-p3 0.9961 0.9922 1 / 0.034)",
    mauveA3: "color(display-p3 0.9373 0.9333 0.9922 / 0.077)",
    mauveA4: "color(display-p3 0.9569 0.9176 0.9961 / 0.111)",
    mauveA5: "color(display-p3 0.9686 0.9373 1 / 0.14)",
    mauveA6: "color(display-p3 0.9529 0.9255 0.9961 / 0.183)",
    mauveA7: "color(display-p3 0.9451 0.9294 1 / 0.246)",
    mauveA8: "color(display-p3 0.9373 0.9176 1 / 0.361)",
    mauveA9: "color(display-p3 0.9333 0.9176 1 / 0.424)",
    mauveA10: "color(display-p3 0.9412 0.9255 1 / 0.479)",
    mauveA11: "color(display-p3 0.9569 0.9451 1 / 0.717)",
    mauveA12: "color(display-p3 0.9922 0.9922 1 / 0.937)",
};
const slateDark = {
    slate1: "#111113",
    slate2: "#18191b",
    slate3: "#212225",
    slate4: "#272a2d",
    slate5: "#2e3135",
    slate6: "#363a3f",
    slate7: "#43484e",
    slate8: "#5a6169",
    slate9: "#696e77",
    slate10: "#777b84",
    slate11: "#afb3ba",
    slate12: "#edeef0",
};
const slateDarkA = {
    slateA1: "#00000000",
    slateA2: "#d8f4f609",
    slateA3: "#ddeaf814",
    slateA4: "#d3edf81d",
    slateA5: "#d9edfe25",
    slateA6: "#d6ebfd30",
    slateA7: "#d9edff40",
    slateA8: "#d9edff5d",
    slateA9: "#dfebfd6d",
    slateA10: "#e5edfd7b",
    slateA11: "#f0f5feb5",
    slateA12: "#fcfdffef",
};
const slateDarkP3 = {
    slate1: "color(display-p3 0.0667 0.0667 0.0738)",
    slate2: "color(display-p3 0.0948 0.0979 0.1051)",
    slate3: "color(display-p3 0.1299 0.1348 0.1451)",
    slate4: "color(display-p3 0.1564 0.1629 0.1755)",
    slate5: "color(display-p3 0.1826 0.1909 0.2058)",
    slate6: "color(display-p3 0.2155 0.2263 0.2439)",
    slate7: "color(display-p3 0.2652 0.2802 0.3018)",
    slate8: "color(display-p3 0.3571 0.381 0.4095)",
    slate9: "color(display-p3 0.4153 0.4307 0.4633)",
    slate10: "color(display-p3 0.4683 0.483 0.5141)",
    slate11: "color(display-p3 0.6903 0.7021 0.7259)",
    slate12: "color(display-p3 0.9301 0.9332 0.9404)",
};
const slateDarkP3A = {
    slateA1: "color(display-p3 0 0 0 / 0)",
    slateA2: "color(display-p3 0.8745 0.9922 1 / 0.034)",
    slateA3: "color(display-p3 0.8824 0.9333 0.9922 / 0.077)",
    slateA4: "color(display-p3 0.8824 0.9529 0.9961 / 0.111)",
    slateA5: "color(display-p3 0.9098 0.9647 1 / 0.14)",
    slateA6: "color(display-p3 0.8824 0.949 0.9961 / 0.183)",
    slateA7: "color(display-p3 0.8824 0.9294 1 / 0.246)",
    slateA8: "color(display-p3 0.8706 0.9373 1 / 0.361)",
    slateA9: "color(display-p3 0.898 0.9373 1 / 0.42)",
    slateA10: "color(display-p3 0.9098 0.9451 1 / 0.475)",
    slateA11: "color(display-p3 0.9529 0.9686 0.9961 / 0.704)",
    slateA12: "color(display-p3 0.9882 0.9922 1 / 0.937)",
};
const sageDark = {
    sage1: "#101211",
    sage2: "#171918",
    sage3: "#202221",
    sage4: "#272a29",
    sage5: "#2e3130",
    sage6: "#373b39",
    sage7: "#444947",
    sage8: "#5b625f",
    sage9: "#63706b",
    sage10: "#717d78",
    sage11: "#acb4b0",
    sage12: "#eceeed",
};
const sageDarkA = {
    sageA1: "#00000000",
    sageA2: "#f0f2f108",
    sageA3: "#f3f5f412",
    sageA4: "#f2fefd1a",
    sageA5: "#f1fbfa22",
    sageA6: "#edfbf42d",
    sageA7: "#edfcf73c",
    sageA8: "#ebfdf657",
    sageA9: "#dffdf266",
    sageA10: "#e5fdf474",
    sageA11: "#f4fef9af",
    sageA12: "#fdfffeed",
};
const sageDarkP3 = {
    sage1: "color(display-p3 0.0642 0.0703 0.0669)",
    sage2: "color(display-p3 0.0916 0.0978 0.0943)",
    sage3: "color(display-p3 0.1276 0.1349 0.1309)",
    sage4: "color(display-p3 0.1551 0.1637 0.1591)",
    sage5: "color(display-p3 0.1829 0.193 0.1877)",
    sage6: "color(display-p3 0.2177 0.2299 0.2237)",
    sage7: "color(display-p3 0.2695 0.2846 0.2771)",
    sage8: "color(display-p3 0.3618 0.3821 0.3727)",
    sage9: "color(display-p3 0.3979 0.4376 0.4206)",
    sage10: "color(display-p3 0.4522 0.4891 0.4727)",
    sage11: "color(display-p3 0.6801 0.7045 0.6923)",
    sage12: "color(display-p3 0.9269 0.9331 0.9296)",
};
const sageDarkP3A = {
    sageA1: "color(display-p3 0 0 0 / 0)",
    sageA2: "color(display-p3 0.9765 0.9882 0.9843 / 0.03)",
    sageA3: "color(display-p3 0.9922 0.9451 0.9412 / 0.072)",
    sageA4: "color(display-p3 0.9882 0.9961 0.9922 / 0.102)",
    sageA5: "color(display-p3 0.9922 1 0.9961 / 0.131)",
    sageA6: "color(display-p3 0.9725 1 0.9765 / 0.173)",
    sageA7: "color(display-p3 0.9569 1 0.9765 / 0.233)",
    sageA8: "color(display-p3 0.9569 1 0.9843 / 0.334)",
    sageA9: "color(display-p3 0.902 1 0.9569 / 0.397)",
    sageA10: "color(display-p3 0.9216 1 0.9725 / 0.452)",
    sageA11: "color(display-p3 0.9647 1 0.9843 / 0.684)",
    sageA12: "color(display-p3 0.9922 1 0.9961 / 0.929)",
};
const oliveDark = {
    olive1: "#111210",
    olive2: "#181917",
    olive3: "#212220",
    olive4: "#282a27",
    olive5: "#2f312e",
    olive6: "#383a36",
    olive7: "#454843",
    olive8: "#5c625b",
    olive9: "#687066",
    olive10: "#767d74",
    olive11: "#afb5ad",
    olive12: "#eceeec",
};
const oliveDarkA = {
    oliveA1: "#00000000",
    oliveA2: "#f1f2f008",
    oliveA3: "#f4f5f312",
    oliveA4: "#f3fef21a",
    oliveA5: "#f2fbf122",
    oliveA6: "#f4faed2c",
    oliveA7: "#f2fced3b",
    oliveA8: "#edfdeb57",
    oliveA9: "#ebfde766",
    oliveA10: "#f0fdec74",
    oliveA11: "#f6fef4b0",
    oliveA12: "#fdfffded",
};
const oliveDarkP3 = {
    olive1: "color(display-p3 0.0674 0.0705 0.0634)",
    olive2: "color(display-p3 0.0948 0.0979 0.0909)",
    olive3: "color(display-p3 0.1309 0.135 0.1264)",
    olive4: "color(display-p3 0.1582 0.1633 0.1532)",
    olive5: "color(display-p3 0.1858 0.1922 0.1803)",
    olive6: "color(display-p3 0.2207 0.2289 0.2146)",
    olive7: "color(display-p3 0.2726 0.2837 0.266)",
    olive8: "color(display-p3 0.3653 0.3823 0.3591)",
    olive9: "color(display-p3 0.4142 0.4379 0.4041)",
    olive10: "color(display-p3 0.4673 0.4903 0.4576)",
    olive11: "color(display-p3 0.6898 0.7094 0.6821)",
    olive12: "color(display-p3 0.9269 0.9331 0.9261)",
};
const oliveDarkP3A = {
    oliveA1: "color(display-p3 0 0 0 / 0)",
    oliveA2: "color(display-p3 0.9843 0.9882 0.9765 / 0.03)",
    oliveA3: "color(display-p3 0.9922 0.9961 0.9882 / 0.068)",
    oliveA4: "color(display-p3 0.9529 0.9961 0.949 / 0.102)",
    oliveA5: "color(display-p3 0.9686 1 0.9647 / 0.131)",
    oliveA6: "color(display-p3 0.9725 1 0.9686 / 0.169)",
    oliveA7: "color(display-p3 0.9804 1 0.9608 / 0.228)",
    oliveA8: "color(display-p3 0.9608 1 0.9569 / 0.334)",
    oliveA9: "color(display-p3 0.949 1 0.9216 / 0.397)",
    oliveA10: "color(display-p3 0.9529 1 0.9412 / 0.452)",
    oliveA11: "color(display-p3 0.9765 1 0.9647 / 0.688)",
    oliveA12: "color(display-p3 0.9922 1 0.9922 / 0.929)",
};
const sandDark = {
    sand1: "#111110",
    sand2: "#191918",
    sand3: "#222221",
    sand4: "#2a2a28",
    sand5: "#31312e",
    sand6: "#3b3a37",
    sand7: "#494844",
    sand8: "#62605b",
    sand9: "#6f6d66",
    sand10: "#7c7a73",
    sand11: "#b4b2ac",
    sand12: "#eeeeec",
};
const sandDarkA = {
    sandA1: "#00000000",
    sandA2: "#f4f4f309",
    sandA3: "#f6f6f513",
    sandA4: "#fefef31b",
    sandA5: "#fbfbeb23",
    sandA6: "#fffaed2d",
    sandA7: "#fffbed3c",
    sandA8: "#fff9eb57",
    sandA9: "#fffae965",
    sandA10: "#fffaec73",
    sandA11: "#fffcf4af",
    sandA12: "#fffffded",
};
const sandDarkP3 = {
    sand1: "color(display-p3 0.0667 0.0667 0.0631)",
    sand2: "color(display-p3 0.098 0.098 0.0945)",
    sand3: "color(display-p3 0.1351 0.1349 0.1291)",
    sand4: "color(display-p3 0.1639 0.1634 0.1559)",
    sand5: "color(display-p3 0.1932 0.1923 0.183)",
    sand6: "color(display-p3 0.2301 0.2286 0.217)",
    sand7: "color(display-p3 0.285 0.2824 0.2675)",
    sand8: "color(display-p3 0.3836 0.3781 0.3574)",
    sand9: "color(display-p3 0.4339 0.4277 0.4027)",
    sand10: "color(display-p3 0.4859 0.4801 0.4552)",
    sand11: "color(display-p3 0.7036 0.6995 0.6759)",
    sand12: "color(display-p3 0.9333 0.9333 0.9262)",
};
const sandDarkP3A = {
    sandA1: "color(display-p3 0 0 0 / 0)",
    sandA2: "color(display-p3 0.9922 0.9922 0.9882 / 0.034)",
    sandA3: "color(display-p3 0.9961 0.9961 0.9922 / 0.072)",
    sandA4: "color(display-p3 0.9922 0.9922 0.9529 / 0.106)",
    sandA5: "color(display-p3 1 1 0.9647 / 0.135)",
    sandA6: "color(display-p3 1 0.9765 0.9294 / 0.177)",
    sandA7: "color(display-p3 1 0.9843 0.9294 / 0.236)",
    sandA8: "color(display-p3 1 0.9765 0.9255 / 0.341)",
    sandA9: "color(display-p3 1 0.9804 0.9255 / 0.395)",
    sandA10: "color(display-p3 1 0.9843 0.9333 / 0.45)",
    sandA11: "color(display-p3 1 0.9961 0.9608 / 0.681)",
    sandA12: "color(display-p3 1 1 0.9922 / 0.929)",
};
const tomatoDark = {
    tomato1: "#181111",
    tomato2: "#1f1513",
    tomato3: "#391714",
    tomato4: "#4e1511",
    tomato5: "#5e1c16",
    tomato6: "#6e2920",
    tomato7: "#853a2d",
    tomato8: "#ac4d39",
    tomato9: "#e54d2e",
    tomato10: "#ec6142",
    tomato11: "#ff977d",
    tomato12: "#fbd3cb",
};
const tomatoDarkA = {
    tomatoA1: "#f1121208",
    tomatoA2: "#ff55330f",
    tomatoA3: "#ff35232b",
    tomatoA4: "#fd201142",
    tomatoA5: "#fe332153",
    tomatoA6: "#ff4f3864",
    tomatoA7: "#fd644a7d",
    tomatoA8: "#fe6d4ea7",
    tomatoA9: "#fe5431e4",
    tomatoA10: "#ff6847eb",
    tomatoA11: "#ff977d",
    tomatoA12: "#ffd6cefb",
};
const tomatoDarkP3 = {
    tomato1: "color(display-p3 0.0898 0.0677 0.0672)",
    tomato2: "color(display-p3 0.1155 0.0839 0.0761)",
    tomato3: "color(display-p3 0.2051 0.0972 0.0833)",
    tomato4: "color(display-p3 0.2819 0.0992 0.0771)",
    tomato5: "color(display-p3 0.3386 0.1292 0.1015)",
    tomato6: "color(display-p3 0.3982 0.1793 0.1414)",
    tomato7: "color(display-p3 0.4868 0.2449 0.1937)",
    tomato8: "color(display-p3 0.6285 0.3218 0.248)",
    tomato9: "color(display-p3 0.8309 0.3454 0.2311)",
    tomato10: "color(display-p3 0.862 0.415 0.2979)",
    tomato11: "color(display-p3 1 0.585 0.455)",
    tomato12: "color(display-p3 0.9589 0.8333 0.8021)",
};
const tomatoDarkP3A = {
    tomatoA1: "color(display-p3 0.9725 0.0706 0.0706 / 0.026)",
    tomatoA2: "color(display-p3 0.9922 0.3765 0.2235 / 0.051)",
    tomatoA3: "color(display-p3 0.9961 0.2824 0.1765 / 0.148)",
    tomatoA4: "color(display-p3 1 0.2039 0.1176 / 0.232)",
    tomatoA5: "color(display-p3 1 0.2863 0.1922 / 0.29)",
    tomatoA6: "color(display-p3 1 0.3882 0.2784 / 0.358)",
    tomatoA7: "color(display-p3 1 0.4588 0.349 / 0.45)",
    tomatoA8: "color(display-p3 1 0.4902 0.3686 / 0.601)",
    tomatoA9: "color(display-p3 1 0.4078 0.2667 / 0.82)",
    tomatoA10: "color(display-p3 1 0.4784 0.3412 / 0.853)",
    tomatoA11: "color(display-p3 1 0.585 0.455)",
    tomatoA12: "color(display-p3 0.9589 0.8333 0.8021)",
};
const redDark = {
    red1: "#191111",
    red2: "#201314",
    red3: "#3b1219",
    red4: "#500f1c",
    red5: "#611623",
    red6: "#72232d",
    red7: "#8c333a",
    red8: "#b54548",
    red9: "#e5484d",
    red10: "#ec5d5e",
    red11: "#ff9592",
    red12: "#ffd1d9",
};
const redDarkA = {
    redA1: "#f4121209",
    redA2: "#f22f3e11",
    redA3: "#ff173f2d",
    redA4: "#fe0a3b44",
    redA5: "#ff204756",
    redA6: "#ff3e5668",
    redA7: "#ff536184",
    redA8: "#ff5d61b0",
    redA9: "#fe4e54e4",
    redA10: "#ff6465eb",
    redA11: "#ff9592",
    redA12: "#ffd1d9",
};
const redDarkP3 = {
    red1: "color(display-p3 0.0931 0.0679 0.0673)",
    red2: "color(display-p3 0.1176 0.0771 0.0791)",
    red3: "color(display-p3 0.2105 0.0806 0.0991)",
    red4: "color(display-p3 0.2872 0.0793 0.1131)",
    red5: "color(display-p3 0.3477 0.1098 0.1421)",
    red6: "color(display-p3 0.4135 0.1601 0.1827)",
    red7: "color(display-p3 0.5076 0.224 0.2356)",
    red8: "color(display-p3 0.6587 0.2984 0.2974)",
    red9: "color(display-p3 0.83 0.3294 0.3239)",
    red10: "color(display-p3 0.8609 0.4026 0.3868)",
    red11: "color(display-p3 1 0.57 0.55)",
    red12: "color(display-p3 0.9712 0.8264 0.8516)",
};
const redDarkP3A = {
    redA1: "color(display-p3 0.9843 0.0706 0.0706 / 0.03)",
    redA2: "color(display-p3 0.9961 0.2824 0.2824 / 0.055)",
    redA3: "color(display-p3 1 0.1686 0.2706 / 0.156)",
    redA4: "color(display-p3 1 0.1176 0.2667 / 0.236)",
    redA5: "color(display-p3 1 0.2118 0.3137 / 0.303)",
    redA6: "color(display-p3 1 0.3216 0.3843 / 0.37)",
    redA7: "color(display-p3 1 0.4 0.4275 / 0.471)",
    redA8: "color(display-p3 1 0.4314 0.4314 / 0.635)",
    redA9: "color(display-p3 1 0.3882 0.3843 / 0.82)",
    redA10: "color(display-p3 1 0.4627 0.4471 / 0.853)",
    redA11: "color(display-p3 1 0.57 0.55)",
    redA12: "color(display-p3 0.9712 0.8264 0.8516)",
};
const rubyDark = {
    ruby1: "#191113",
    ruby2: "#1e1517",
    ruby3: "#3a141e",
    ruby4: "#4e1326",
    ruby5: "#5e192e",
    ruby6: "#6f2539",
    ruby7: "#883447",
    ruby8: "#b3445a",
    ruby9: "#e54666",
    ruby10: "#ec5a72",
    ruby11: "#ff949d",
    ruby12: "#fed2e1",
};
const rubyDarkA = {
    rubyA1: "#f4124a09",
    rubyA2: "#fe5a7f0e",
    rubyA3: "#ff235d2c",
    rubyA4: "#fd196242",
    rubyA5: "#fe2a6b53",
    rubyA6: "#ff447665",
    rubyA7: "#ff577d80",
    rubyA8: "#ff5c7cae",
    rubyA9: "#fe4c70e4",
    rubyA10: "#ff617beb",
    rubyA11: "#ff949d",
    rubyA12: "#ffd3e2fe",
};
const rubyDarkP3 = {
    ruby1: "color(display-p3 0.0931 0.0679 0.0744)",
    ruby2: "color(display-p3 0.1131 0.0825 0.0892)",
    ruby3: "color(display-p3 0.2097 0.0888 0.1176)",
    ruby4: "color(display-p3 0.2813 0.0911 0.1477)",
    ruby5: "color(display-p3 0.3388 0.1197 0.1808)",
    ruby6: "color(display-p3 0.4025 0.1654 0.2234)",
    ruby7: "color(display-p3 0.4955 0.2243 0.2811)",
    ruby8: "color(display-p3 0.6519 0.2953 0.3585)",
    ruby9: "color(display-p3 0.8296 0.3231 0.4084)",
    ruby10: "color(display-p3 0.8571 0.3917 0.4552)",
    ruby11: "color(display-p3 1 0.57 0.59)",
    ruby12: "color(display-p3 0.9684 0.83 0.8804)",
};
const rubyDarkP3A = {
    rubyA1: "color(display-p3 0.9843 0.0706 0.3294 / 0.03)",
    rubyA2: "color(display-p3 0.9922 0.3765 0.5294 / 0.051)",
    rubyA3: "color(display-p3 0.9961 0.2235 0.4039 / 0.152)",
    rubyA4: "color(display-p3 1 0.1686 0.4235 / 0.232)",
    rubyA5: "color(display-p3 1 0.2588 0.4588 / 0.29)",
    rubyA6: "color(display-p3 1 0.3412 0.502 / 0.362)",
    rubyA7: "color(display-p3 1 0.4118 0.5412 / 0.458)",
    rubyA8: "color(display-p3 1 0.4314 0.5333 / 0.627)",
    rubyA9: "color(display-p3 1 0.3765 0.4824 / 0.82)",
    rubyA10: "color(display-p3 1 0.4471 0.5216 / 0.849)",
    rubyA11: "color(display-p3 1 0.57 0.59)",
    rubyA12: "color(display-p3 0.9684 0.83 0.8804)",
};
const crimsonDark = {
    crimson1: "#191114",
    crimson2: "#201318",
    crimson3: "#381525",
    crimson4: "#4d122f",
    crimson5: "#5c1839",
    crimson6: "#6d2545",
    crimson7: "#873356",
    crimson8: "#b0436e",
    crimson9: "#e93d82",
    crimson10: "#ee518a",
    crimson11: "#ff92ad",
    crimson12: "#fdd3e8",
};
const crimsonDarkA = {
    crimsonA1: "#f4126709",
    crimsonA2: "#f22f7a11",
    crimsonA3: "#fe2a8b2a",
    crimsonA4: "#fd158741",
    crimsonA5: "#fd278f51",
    crimsonA6: "#fe459763",
    crimsonA7: "#fd559b7f",
    crimsonA8: "#fe5b9bab",
    crimsonA9: "#fe418de8",
    crimsonA10: "#ff5693ed",
    crimsonA11: "#ff92ad",
    crimsonA12: "#ffd5eafd",
};
const crimsonDarkP3 = {
    crimson1: "color(display-p3 0.0931 0.0679 0.078)",
    crimson2: "color(display-p3 0.1168 0.078 0.0952)",
    crimson3: "color(display-p3 0.2032 0.0913 0.1432)",
    crimson4: "color(display-p3 0.2766 0.0873 0.1819)",
    crimson5: "color(display-p3 0.3322 0.1152 0.2202)",
    crimson6: "color(display-p3 0.3943 0.1625 0.2678)",
    crimson7: "color(display-p3 0.4888 0.2224 0.3355)",
    crimson8: "color(display-p3 0.6384 0.2892 0.4288)",
    crimson9: "color(display-p3 0.8425 0.2978 0.507)",
    crimson10: "color(display-p3 0.8639 0.3639 0.5391)",
    crimson11: "color(display-p3 1 0.56 0.66)",
    crimson12: "color(display-p3 0.9656 0.8336 0.9057)",
};
const crimsonDarkP3A = {
    crimsonA1: "color(display-p3 0.9843 0.0706 0.4627 / 0.03)",
    crimsonA2: "color(display-p3 0.9961 0.2824 0.5686 / 0.055)",
    crimsonA3: "color(display-p3 0.9961 0.2275 0.6 / 0.148)",
    crimsonA4: "color(display-p3 1 0.1569 0.5686 / 0.227)",
    crimsonA5: "color(display-p3 1 0.2314 0.6039 / 0.286)",
    crimsonA6: "color(display-p3 1 0.3333 0.6353 / 0.353)",
    crimsonA7: "color(display-p3 1 0.4157 0.6627 / 0.454)",
    crimsonA8: "color(display-p3 0.9961 0.4275 0.651 / 0.614)",
    crimsonA9: "color(display-p3 1 0.3451 0.5961 / 0.832)",
    crimsonA10: "color(display-p3 1 0.4196 0.6196 / 0.853)",
    crimsonA11: "color(display-p3 1 0.56 0.66)",
    crimsonA12: "color(display-p3 0.9656 0.8336 0.9057)",
};
const pinkDark = {
    pink1: "#191117",
    pink2: "#21121d",
    pink3: "#37172f",
    pink4: "#4b143d",
    pink5: "#591c47",
    pink6: "#692955",
    pink7: "#833869",
    pink8: "#a84885",
    pink9: "#d6409f",
    pink10: "#de51a8",
    pink11: "#ff8dcc",
    pink12: "#fdd1ea",
};
const pinkDarkA = {
    pinkA1: "#f412bc09",
    pinkA2: "#f420bb12",
    pinkA3: "#fe37cc29",
    pinkA4: "#fc1ec43f",
    pinkA5: "#fd35c24e",
    pinkA6: "#fd51c75f",
    pinkA7: "#fd62c87b",
    pinkA8: "#ff68c8a2",
    pinkA9: "#fe49bcd4",
    pinkA10: "#ff5cc0dc",
    pinkA11: "#ff8dcc",
    pinkA12: "#ffd3ecfd",
};
const pinkDarkP3 = {
    pink1: "color(display-p3 0.0931 0.0679 0.0888)",
    pink2: "color(display-p3 0.1214 0.0727 0.1098)",
    pink3: "color(display-p3 0.1983 0.0983 0.1792)",
    pink4: "color(display-p3 0.2708 0.0952 0.231)",
    pink5: "color(display-p3 0.3201 0.1274 0.2728)",
    pink6: "color(display-p3 0.3816 0.1775 0.3258)",
    pink7: "color(display-p3 0.4773 0.2384 0.405)",
    pink8: "color(display-p3 0.6119 0.3044 0.5102)",
    pink9: "color(display-p3 0.7749 0.2973 0.6104)",
    pink10: "color(display-p3 0.8084 0.3563 0.6449)",
    pink11: "color(display-p3 1 0.535 0.78)",
    pink12: "color(display-p3 0.9645 0.8261 0.9124)",
};
const pinkDarkP3A = {
    pinkA1: "color(display-p3 0.9843 0.0706 0.8549 / 0.03)",
    pinkA2: "color(display-p3 1 0.2 0.8 / 0.059)",
    pinkA3: "color(display-p3 1 0.2863 0.8627 / 0.143)",
    pinkA4: "color(display-p3 1 0.1922 0.8196 / 0.219)",
    pinkA5: "color(display-p3 1 0.2824 0.8275 / 0.274)",
    pinkA6: "color(display-p3 1 0.3961 0.8353 / 0.337)",
    pinkA7: "color(display-p3 1 0.4588 0.8314 / 0.442)",
    pinkA8: "color(display-p3 1 0.4784 0.8275 / 0.585)",
    pinkA9: "color(display-p3 1 0.3725 0.7843 / 0.761)",
    pinkA10: "color(display-p3 1 0.4353 0.7922 / 0.795)",
    pinkA11: "color(display-p3 1 0.535 0.78)",
    pinkA12: "color(display-p3 0.9645 0.8261 0.9124)",
};
const plumDark = {
    plum1: "#181118",
    plum2: "#201320",
    plum3: "#351a35",
    plum4: "#451d47",
    plum5: "#512454",
    plum6: "#5e3061",
    plum7: "#734079",
    plum8: "#92549c",
    plum9: "#ab4aba",
    plum10: "#b658c4",
    plum11: "#e796f3",
    plum12: "#f4d4f4",
};
const plumDarkA = {
    plumA1: "#f112f108",
    plumA2: "#f22ff211",
    plumA3: "#fd4cfd27",
    plumA4: "#f646ff3a",
    plumA5: "#f455ff48",
    plumA6: "#f66dff56",
    plumA7: "#f07cfd70",
    plumA8: "#ee84ff95",
    plumA9: "#e961feb6",
    plumA10: "#ed70ffc0",
    plumA11: "#f19cfef3",
    plumA12: "#feddfef4",
};
const plumDarkP3 = {
    plum1: "color(display-p3 0.0898 0.0677 0.0924)",
    plum2: "color(display-p3 0.1176 0.0771 0.1212)",
    plum3: "color(display-p3 0.1918 0.1053 0.2021)",
    plum4: "color(display-p3 0.2503 0.1215 0.271)",
    plum5: "color(display-p3 0.2934 0.152 0.3185)",
    plum6: "color(display-p3 0.3435 0.1984 0.372)",
    plum7: "color(display-p3 0.4243 0.2623 0.4614)",
    plum8: "color(display-p3 0.5399 0.3408 0.5954)",
    plum9: "color(display-p3 0.6238 0.3128 0.7077)",
    plum10: "color(display-p3 0.6657 0.3653 0.7476)",
    plum11: "color(display-p3 0.8599 0.602 0.9329)",
    plum12: "color(display-p3 0.9362 0.8359 0.9485)",
};
const plumDarkP3A = {
    plumA1: "color(display-p3 0.8549 0.0706 0.9843 / 0.03)",
    plumA2: "color(display-p3 0.9333 0.2667 1 / 0.059)",
    plumA3: "color(display-p3 0.9176 0.3333 0.9961 / 0.148)",
    plumA4: "color(display-p3 0.9098 0.3176 1 / 0.219)",
    plumA5: "color(display-p3 0.9137 0.3882 1 / 0.269)",
    plumA6: "color(display-p3 0.9176 0.4745 1 / 0.328)",
    plumA7: "color(display-p3 0.9059 0.5294 1 / 0.425)",
    plumA8: "color(display-p3 0.9059 0.5529 1 / 0.568)",
    plumA9: "color(display-p3 0.8824 0.4275 1 / 0.685)",
    plumA10: "color(display-p3 0.8863 0.4706 0.9961 / 0.732)",
    plumA11: "color(display-p3 0.8599 0.602 0.9329)",
    plumA12: "color(display-p3 0.9362 0.8359 0.9485)",
};
const purpleDark = {
    purple1: "#18111b",
    purple2: "#1e1423",
    purple3: "#301c3b",
    purple4: "#3d224e",
    purple5: "#47295c",
    purple6: "#54336b",
    purple7: "#664282",
    purple8: "#8457aa",
    purple9: "#8e4ec6",
    purple10: "#9a5cd0",
    purple11: "#d19dff",
    purple12: "#ecd9fa",
};
const purpleDarkA = {
    purpleA1: "#b412f90b",
    purpleA2: "#b738f714",
    purpleA3: "#c150ff2d",
    purpleA4: "#bb53fd42",
    purpleA5: "#bb5cfd51",
    purpleA6: "#c16afd61",
    purpleA7: "#c378fd7a",
    purpleA8: "#c47effa4",
    purpleA9: "#b661ffc2",
    purpleA10: "#bc6fffcd",
    purpleA11: "#d19dff",
    purpleA12: "#f1ddfffa",
};
const purpleDarkP3 = {
    purple1: "color(display-p3 0.0898 0.0677 0.1033)",
    purple2: "color(display-p3 0.1116 0.0814 0.1325)",
    purple3: "color(display-p3 0.1752 0.1122 0.224)",
    purple4: "color(display-p3 0.2234 0.1364 0.2964)",
    purple5: "color(display-p3 0.2634 0.1669 0.3481)",
    purple6: "color(display-p3 0.3105 0.2074 0.4057)",
    purple7: "color(display-p3 0.3808 0.2653 0.4957)",
    purple8: "color(display-p3 0.4905 0.3492 0.6491)",
    purple9: "color(display-p3 0.5231 0.3183 0.7512)",
    purple10: "color(display-p3 0.5704 0.3728 0.7911)",
    purple11: "color(display-p3 0.8 0.62 1)",
    purple12: "color(display-p3 0.9128 0.8536 0.9709)",
};
const purpleDarkP3A = {
    purpleA1: "color(display-p3 0.6863 0.0706 0.9961 / 0.038)",
    purpleA2: "color(display-p3 0.6667 0.2863 0.9961 / 0.072)",
    purpleA3: "color(display-p3 0.7176 0.349 0.9961 / 0.169)",
    purpleA4: "color(display-p3 0.702 0.3529 1 / 0.248)",
    purpleA5: "color(display-p3 0.7176 0.4039 1 / 0.303)",
    purpleA6: "color(display-p3 0.7412 0.4588 1 / 0.362)",
    purpleA7: "color(display-p3 0.7529 0.5059 1 / 0.458)",
    purpleA8: "color(display-p3 0.7451 0.5176 1 / 0.627)",
    purpleA9: "color(display-p3 0.6863 0.4078 1 / 0.736)",
    purpleA10: "color(display-p3 0.7098 0.4588 1 / 0.778)",
    purpleA11: "color(display-p3 0.8 0.62 1)",
    purpleA12: "color(display-p3 0.9128 0.8536 0.9709)",
};
const violetDark = {
    violet1: "#14121f",
    violet2: "#1b1525",
    violet3: "#291f44",
    violet4: "#33255b",
    violet5: "#3c2e69",
    violet6: "#473876",
    violet7: "#56468b",
    violet8: "#6958ad",
    violet9: "#6e56cf",
    violet10: "#7d66d9",
    violet11: "#baa7ff",
    violet12: "#e2ddfe",
};
const violetDarkA = {
    violetA1: "#4422ff0f",
    violetA2: "#853ff916",
    violetA3: "#8152fe37",
    violetA4: "#7d51fd50",
    violetA5: "#845ffd5f",
    violetA6: "#8f6cfd6d",
    violetA7: "#9879ff83",
    violetA8: "#977dfea8",
    violetA9: "#8668ffcc",
    violetA10: "#9176fed7",
    violetA11: "#baa7ff",
    violetA12: "#e3defffe",
};
const violetDarkP3 = {
    violet1: "color(display-p3 0.0771 0.0709 0.1179)",
    violet2: "color(display-p3 0.1013 0.0847 0.1418)",
    violet3: "color(display-p3 0.156 0.125 0.2595)",
    violet4: "color(display-p3 0.191 0.1486 0.3451)",
    violet5: "color(display-p3 0.2268 0.1818 0.3967)",
    violet6: "color(display-p3 0.2696 0.2232 0.4489)",
    violet7: "color(display-p3 0.3263 0.2773 0.5299)",
    violet8: "color(display-p3 0.3992 0.3459 0.6558)",
    violet9: "color(display-p3 0.4166 0.3409 0.7843)",
    violet10: "color(display-p3 0.477 0.4024 0.8226)",
    violet11: "color(display-p3 0.72 0.65 1)",
    violet12: "color(display-p3 0.8828 0.8673 0.9857)",
};
const violetDarkP3A = {
    violetA1: "color(display-p3 0.2824 0.1412 0.9961 / 0.055)",
    violetA2: "color(display-p3 0.5098 0.3137 1 / 0.08)",
    violetA3: "color(display-p3 0.5059 0.3529 1 / 0.206)",
    violetA4: "color(display-p3 0.4902 0.3451 1 / 0.299)",
    violetA5: "color(display-p3 0.5255 0.3922 1 / 0.353)",
    violetA6: "color(display-p3 0.5686 0.4549 1 / 0.408)",
    violetA7: "color(display-p3 0.5882 0.4941 1 / 0.496)",
    violetA8: "color(display-p3 0.5961 0.5098 1 / 0.631)",
    violetA9: "color(display-p3 0.5216 0.4235 1 / 0.769)",
    violetA10: "color(display-p3 0.5765 0.4824 1 / 0.811)",
    violetA11: "color(display-p3 0.72 0.65 1)",
    violetA12: "color(display-p3 0.8828 0.8673 0.9857)",
};
const irisDark = {
    iris1: "#13131e",
    iris2: "#171625",
    iris3: "#202248",
    iris4: "#262a65",
    iris5: "#303374",
    iris6: "#3d3e82",
    iris7: "#4a4a95",
    iris8: "#5958b1",
    iris9: "#5b5bd6",
    iris10: "#6e6ade",
    iris11: "#b1a9ff",
    iris12: "#e0dffe",
};
const irisDarkA = {
    irisA1: "#3636fe0e",
    irisA2: "#564bf916",
    irisA3: "#525bff3b",
    irisA4: "#4d58ff5a",
    irisA5: "#5b62fd6b",
    irisA6: "#6d6ffd7a",
    irisA7: "#7777fe8e",
    irisA8: "#7b7afeac",
    irisA9: "#6a6afed4",
    irisA10: "#7d79ffdc",
    irisA11: "#b1a9ff",
    irisA12: "#e1e0fffe",
};
const irisDarkP3 = {
    iris1: "color(display-p3 0.0745 0.0745 0.1144)",
    iris2: "color(display-p3 0.0886 0.0863 0.1396)",
    iris3: "color(display-p3 0.1285 0.1346 0.2732)",
    iris4: "color(display-p3 0.1525 0.1645 0.3819)",
    iris5: "color(display-p3 0.192 0.2006 0.44)",
    iris6: "color(display-p3 0.2386 0.2411 0.4913)",
    iris7: "color(display-p3 0.291 0.2892 0.5651)",
    iris8: "color(display-p3 0.3496 0.345 0.673)",
    iris9: "color(display-p3 0.3569 0.3569 0.8104)",
    iris10: "color(display-p3 0.4278 0.416 0.8435)",
    iris11: "color(display-p3 0.685 0.662 1)",
    iris12: "color(display-p3 0.8777 0.8746 0.9861)",
};
const irisDarkP3A = {
    irisA1: "color(display-p3 0.2235 0.2235 0.9922 / 0.051)",
    irisA2: "color(display-p3 0.3608 0.3137 1 / 0.08)",
    irisA3: "color(display-p3 0.349 0.3686 1 / 0.223)",
    irisA4: "color(display-p3 0.3255 0.3608 1 / 0.337)",
    irisA5: "color(display-p3 0.3804 0.4 1 / 0.4)",
    irisA6: "color(display-p3 0.4471 0.4471 1 / 0.454)",
    irisA7: "color(display-p3 0.4863 0.4863 1 / 0.534)",
    irisA8: "color(display-p3 0.502 0.4941 1 / 0.652)",
    irisA9: "color(display-p3 0.4314 0.4314 1 / 0.799)",
    irisA10: "color(display-p3 0.502 0.4863 1 / 0.832)",
    irisA11: "color(display-p3 0.685 0.662 1)",
    irisA12: "color(display-p3 0.8777 0.8746 0.9861)",
};
const indigoDark = {
    indigo1: "#11131f",
    indigo2: "#141726",
    indigo3: "#182449",
    indigo4: "#1d2e62",
    indigo5: "#253974",
    indigo6: "#304384",
    indigo7: "#3a4f97",
    indigo8: "#435db1",
    indigo9: "#3e63dd",
    indigo10: "#5472e4",
    indigo11: "#9eb1ff",
    indigo12: "#d6e1ff",
};
const indigoDarkA = {
    indigoA1: "#1133ff0f",
    indigoA2: "#3354fa17",
    indigoA3: "#2f62ff3c",
    indigoA4: "#3566ff57",
    indigoA5: "#4171fd6b",
    indigoA6: "#5178fd7c",
    indigoA7: "#5a7fff90",
    indigoA8: "#5b81feac",
    indigoA9: "#4671ffdb",
    indigoA10: "#5c7efee3",
    indigoA11: "#9eb1ff",
    indigoA12: "#d6e1ff",
};
const indigoDarkP3 = {
    indigo1: "color(display-p3 0.0681 0.0743 0.118)",
    indigo2: "color(display-p3 0.0815 0.0886 0.1438)",
    indigo3: "color(display-p3 0.1049 0.1412 0.2753)",
    indigo4: "color(display-p3 0.1293 0.1797 0.3691)",
    indigo5: "color(display-p3 0.1627 0.2202 0.439)",
    indigo6: "color(display-p3 0.203 0.2618 0.5001)",
    indigo7: "color(display-p3 0.2446 0.3086 0.575)",
    indigo8: "color(display-p3 0.285 0.362 0.6738)",
    indigo9: "color(display-p3 0.2757 0.3844 0.8369)",
    indigo10: "color(display-p3 0.354 0.4449 0.8659)",
    indigo11: "color(display-p3 0.63 0.69 1)",
    indigo12: "color(display-p3 0.8478 0.8813 0.9896)",
};
const indigoDarkP3A = {
    indigoA1: "color(display-p3 0.0706 0.2118 0.9961 / 0.055)",
    indigoA2: "color(display-p3 0.251 0.3451 0.9882 / 0.085)",
    indigoA3: "color(display-p3 0.2431 0.4039 1 / 0.223)",
    indigoA4: "color(display-p3 0.2627 0.4196 1 / 0.324)",
    indigoA5: "color(display-p3 0.302 0.451 1 / 0.4)",
    indigoA6: "color(display-p3 0.3608 0.4902 1 / 0.467)",
    indigoA7: "color(display-p3 0.3882 0.5098 1 / 0.547)",
    indigoA8: "color(display-p3 0.4039 0.5176 1 / 0.652)",
    indigoA9: "color(display-p3 0.3176 0.451 1 / 0.824)",
    indigoA10: "color(display-p3 0.4039 0.5059 1 / 0.858)",
    indigoA11: "color(display-p3 0.63 0.69 1)",
    indigoA12: "color(display-p3 0.8478 0.8813 0.9896)",
};
const blueDark = {
    blue1: "#0d1520",
    blue2: "#111927",
    blue3: "#0d2847",
    blue4: "#003362",
    blue5: "#004074",
    blue6: "#104d87",
    blue7: "#205d9e",
    blue8: "#2870bd",
    blue9: "#0090ff",
    blue10: "#3b9eff",
    blue11: "#70b8ff",
    blue12: "#c2e6ff",
};
const blueDarkA = {
    blueA1: "#004df211",
    blueA2: "#1166fb18",
    blueA3: "#0077ff3a",
    blueA4: "#0075ff57",
    blueA5: "#0081fd6b",
    blueA6: "#0f89fd7f",
    blueA7: "#2a91fe98",
    blueA8: "#3094feb9",
    blueA9: "#0090ff",
    blueA10: "#3b9eff",
    blueA11: "#70b8ff",
    blueA12: "#c2e6ff",
};
const blueDarkP3 = {
    blue1: "color(display-p3 0.0575 0.0815 0.1219)",
    blue2: "color(display-p3 0.0719 0.0984 0.147)",
    blue3: "color(display-p3 0.0779 0.1539 0.2697)",
    blue4: "color(display-p3 0.0326 0.1969 0.3698)",
    blue5: "color(display-p3 0.0798 0.2448 0.4406)",
    blue6: "color(display-p3 0.1402 0.2983 0.5113)",
    blue7: "color(display-p3 0.1948 0.3608 0.6005)",
    blue8: "color(display-p3 0.2388 0.4342 0.7202)",
    blue9: "color(display-p3 0.2466 0.5561 0.9689)",
    blue10: "color(display-p3 0.3445 0.6116 0.9726)",
    blue11: "color(display-p3 0.49 0.72 1)",
    blue12: "color(display-p3 0.7883 0.8977 0.9898)",
};
const blueDarkP3A = {
    blueA1: "color(display-p3 0 0.3333 1 / 0.059)",
    blueA2: "color(display-p3 0.1137 0.4353 0.9882 / 0.085)",
    blueA3: "color(display-p3 0.1216 0.4627 1 / 0.219)",
    blueA4: "color(display-p3 0 0.4667 1 / 0.324)",
    blueA5: "color(display-p3 0.098 0.5098 1 / 0.4)",
    blueA6: "color(display-p3 0.2235 0.5569 1 / 0.475)",
    blueA7: "color(display-p3 0.2941 0.5843 1 / 0.572)",
    blueA8: "color(display-p3 0.3137 0.5922 1 / 0.702)",
    blueA9: "color(display-p3 0.251 0.5725 0.9961 / 0.967)",
    blueA10: "color(display-p3 0.3569 0.6314 1 / 0.971)",
    blueA11: "color(display-p3 0.49 0.72 1)",
    blueA12: "color(display-p3 0.7883 0.8977 0.9898)",
};
const cyanDark = {
    cyan1: "#0b161a",
    cyan2: "#111b1f",
    cyan3: "#082c36",
    cyan4: "#003848",
    cyan5: "#004557",
    cyan6: "#065468",
    cyan7: "#13677e",
    cyan8: "#11809c",
    cyan9: "#00a2c7",
    cyan10: "#23afd0",
    cyan11: "#4ccce6",
    cyan12: "#b6ecf7",
};
const cyanDarkA = {
    cyanA1: "#0091f70a",
    cyanA2: "#11bbff0f",
    cyanA3: "#00befd28",
    cyanA4: "#00baff3b",
    cyanA5: "#00c2ff4b",
    cyanA6: "#00c7fd5e",
    cyanA7: "#16cdff75",
    cyanA8: "#11cfff95",
    cyanA9: "#00cfffc3",
    cyanA10: "#28d6ffcd",
    cyanA11: "#52e1fee5",
    cyanA12: "#bbf3fef7",
};
const cyanDarkP3 = {
    cyan1: "color(display-p3 0.0531 0.0849 0.0984)",
    cyan2: "color(display-p3 0.0749 0.1048 0.1197)",
    cyan3: "color(display-p3 0.073 0.168 0.2089)",
    cyan4: "color(display-p3 0.0661 0.2162 0.2759)",
    cyan5: "color(display-p3 0.0936 0.2664 0.335)",
    cyan6: "color(display-p3 0.1387 0.3238 0.3989)",
    cyan7: "color(display-p3 0.1875 0.3977 0.4831)",
    cyan8: "color(display-p3 0.2301 0.4956 0.6002)",
    cyan9: "color(display-p3 0.2818 0.6273 0.7647)",
    cyan10: "color(display-p3 0.3313 0.6754 0.8014)",
    cyan11: "color(display-p3 0.446 0.7895 0.887)",
    cyan12: "color(display-p3 0.7571 0.9195 0.9619)",
};
const cyanDarkP3A = {
    cyanA1: "color(display-p3 0 0.6471 0.9922 / 0.034)",
    cyanA2: "color(display-p3 0.2 0.7333 1 / 0.059)",
    cyanA3: "color(display-p3 0.1216 0.7412 0.9961 / 0.152)",
    cyanA4: "color(display-p3 0.0667 0.7373 1 / 0.223)",
    cyanA5: "color(display-p3 0.1647 0.7686 1 / 0.286)",
    cyanA6: "color(display-p3 0.2667 0.7922 1 / 0.358)",
    cyanA7: "color(display-p3 0.3412 0.8078 1 / 0.446)",
    cyanA8: "color(display-p3 0.3569 0.8157 1 / 0.572)",
    cyanA9: "color(display-p3 0.3569 0.8196 1 / 0.748)",
    cyanA10: "color(display-p3 0.4 0.8392 1 / 0.786)",
    cyanA11: "color(display-p3 0.446 0.7895 0.887)",
    cyanA12: "color(display-p3 0.7571 0.9195 0.9619)",
};
const tealDark = {
    teal1: "#0d1514",
    teal2: "#121c1b",
    teal3: "#0d2d2a",
    teal4: "#043b35",
    teal5: "#0b4841",
    teal6: "#16574f",
    teal7: "#1d6960",
    teal8: "#207e73",
    teal9: "#12a594",
    teal10: "#0eb39e",
    teal11: "#0bd8b6",
    teal12: "#adf0dd",
};
const tealDarkA = {
    tealA1: "#00deab05",
    tealA2: "#27fbe60c",
    tealA3: "#00ffe61e",
    tealA4: "#00ffdd2d",
    tealA5: "#00ffe13b",
    tealA6: "#23ffe44b",
    tealA7: "#31fde55f",
    tealA8: "#32ffe775",
    tealA9: "#13ffe49f",
    tealA10: "#0dffe0ae",
    tealA11: "#0afed5d6",
    tealA12: "#b8ffebef",
};
const tealDarkP3 = {
    teal1: "color(display-p3 0.0588 0.0828 0.0789)",
    teal2: "color(display-p3 0.078 0.1096 0.1044)",
    teal3: "color(display-p3 0.0869 0.1755 0.1653)",
    teal4: "color(display-p3 0.0922 0.2272 0.2094)",
    teal5: "color(display-p3 0.1241 0.2771 0.2562)",
    teal6: "color(display-p3 0.1654 0.3348 0.3107)",
    teal7: "color(display-p3 0.2069 0.4056 0.3764)",
    teal8: "color(display-p3 0.2452 0.4888 0.4526)",
    teal9: "color(display-p3 0.297 0.6374 0.5811)",
    teal10: "color(display-p3 0.3195 0.6898 0.6197)",
    teal11: "color(display-p3 0.3881 0.8345 0.7191)",
    teal12: "color(display-p3 0.7343 0.934 0.8696)",
};
const tealDarkP3A = {
    tealA1: "color(display-p3 0 0.9922 0.7608 / 0.017)",
    tealA2: "color(display-p3 0.3176 0.9882 0.902 / 0.047)",
    tealA3: "color(display-p3 0.2353 1 0.898 / 0.118)",
    tealA4: "color(display-p3 0.2275 0.9961 0.8863 / 0.173)",
    tealA5: "color(display-p3 0.3294 1 0.898 / 0.227)",
    tealA6: "color(display-p3 0.4118 1 0.9176 / 0.286)",
    tealA7: "color(display-p3 0.4588 1 0.9255 / 0.362)",
    tealA8: "color(display-p3 0.4667 1 0.9137 / 0.454)",
    tealA9: "color(display-p3 0.4431 0.9961 0.902 / 0.614)",
    tealA10: "color(display-p3 0.4392 0.9961 0.8902 / 0.669)",
    tealA11: "color(display-p3 0.3881 0.8345 0.7191)",
    tealA12: "color(display-p3 0.7343 0.934 0.8696)",
};
const jadeDark = {
    jade1: "#0d1512",
    jade2: "#121c18",
    jade3: "#0f2e22",
    jade4: "#0b3b2c",
    jade5: "#114837",
    jade6: "#1b5745",
    jade7: "#246854",
    jade8: "#2a7e68",
    jade9: "#29a383",
    jade10: "#27b08b",
    jade11: "#1fd8a4",
    jade12: "#adf0d4",
};
const jadeDarkA = {
    jadeA1: "#00de4505",
    jadeA2: "#27fba60c",
    jadeA3: "#02f99920",
    jadeA4: "#00ffaa2d",
    jadeA5: "#11ffb63b",
    jadeA6: "#34ffc24b",
    jadeA7: "#45fdc75e",
    jadeA8: "#48ffcf75",
    jadeA9: "#38feca9d",
    jadeA10: "#31fec7ab",
    jadeA11: "#21fec0d6",
    jadeA12: "#b8ffe1ef",
};
const jadeDarkP3 = {
    jade1: "color(display-p3 0.0588 0.0828 0.0713)",
    jade2: "color(display-p3 0.078 0.1096 0.0938)",
    jade3: "color(display-p3 0.0914 0.1758 0.1376)",
    jade4: "color(display-p3 0.1024 0.2278 0.1765)",
    jade5: "color(display-p3 0.1335 0.2787 0.2214)",
    jade6: "color(display-p3 0.1744 0.3344 0.2731)",
    jade7: "color(display-p3 0.2193 0.4024 0.3354)",
    jade8: "color(display-p3 0.2631 0.4882 0.4115)",
    jade9: "color(display-p3 0.3195 0.6301 0.5205)",
    jade10: "color(display-p3 0.3385 0.6804 0.5551)",
    jade11: "color(display-p3 0.4004 0.8347 0.6561)",
    jade12: "color(display-p3 0.7343 0.934 0.8377)",
};
const jadeDarkP3A = {
    jadeA1: "color(display-p3 0 0.9922 0.298 / 0.017)",
    jadeA2: "color(display-p3 0.3176 0.9882 0.651 / 0.047)",
    jadeA3: "color(display-p3 0.2667 1 0.6667 / 0.118)",
    jadeA4: "color(display-p3 0.2745 0.9961 0.702 / 0.173)",
    jadeA5: "color(display-p3 0.3608 1 0.7412 / 0.227)",
    jadeA6: "color(display-p3 0.4392 1 0.7961 / 0.286)",
    jadeA7: "color(display-p3 0.4902 1 0.8157 / 0.362)",
    jadeA8: "color(display-p3 0.5059 1 0.8353 / 0.45)",
    jadeA9: "color(display-p3 0.4784 0.9961 0.8157 / 0.606)",
    jadeA10: "color(display-p3 0.4745 1 0.8078 / 0.66)",
    jadeA11: "color(display-p3 0.4004 0.8347 0.6561)",
    jadeA12: "color(display-p3 0.7343 0.934 0.8377)",
};
const greenDark = {
    green1: "#0e1512",
    green2: "#131b17",
    green3: "#132d21",
    green4: "#123b29",
    green5: "#184933",
    green6: "#20573e",
    green7: "#28684a",
    green8: "#2f7c57",
    green9: "#30a46c",
    green10: "#33b074",
    green11: "#3dd68c",
    green12: "#b1f1cb",
};
const greenDarkA = {
    greenA1: "#00de4505",
    greenA2: "#40f99d0b",
    greenA3: "#22ff991e",
    greenA4: "#17ff992d",
    greenA5: "#2fffa23c",
    greenA6: "#44ffaa4b",
    greenA7: "#50fdac5e",
    greenA8: "#54ffad73",
    greenA9: "#44ffa49e",
    greenA10: "#43fea4ab",
    greenA11: "#46fea5d4",
    greenA12: "#bbffd7f0",
};
const greenDarkP3 = {
    green1: "color(display-p3 0.0617 0.0832 0.0709)",
    green2: "color(display-p3 0.0809 0.1044 0.0909)",
    green3: "color(display-p3 0.0998 0.1726 0.133)",
    green4: "color(display-p3 0.1164 0.2279 0.1666)",
    green5: "color(display-p3 0.148 0.281 0.2062)",
    green6: "color(display-p3 0.1861 0.3373 0.2498)",
    green7: "color(display-p3 0.2274 0.4023 0.2982)",
    green8: "color(display-p3 0.2696 0.4793 0.3512)",
    green9: "color(display-p3 0.3322 0.6342 0.4416)",
    green10: "color(display-p3 0.3571 0.6816 0.474)",
    green11: "color(display-p3 0.4338 0.8276 0.5733)",
    green12: "color(display-p3 0.7469 0.9381 0.8065)",
};
const greenDarkP3A = {
    greenA1: "color(display-p3 0 0.9922 0.298 / 0.017)",
    greenA2: "color(display-p3 0.4353 0.9804 0.6157 / 0.043)",
    greenA3: "color(display-p3 0.3451 0.9961 0.6549 / 0.114)",
    greenA4: "color(display-p3 0.3647 0.9961 0.6353 / 0.173)",
    greenA5: "color(display-p3 0.4235 1 0.6784 / 0.232)",
    greenA6: "color(display-p3 0.4745 1 0.7059 / 0.29)",
    greenA7: "color(display-p3 0.5137 1 0.7059 / 0.362)",
    greenA8: "color(display-p3 0.5294 1 0.7176 / 0.442)",
    greenA9: "color(display-p3 0.502 0.9961 0.6824 / 0.61)",
    greenA10: "color(display-p3 0.5059 1 0.6824 / 0.66)",
    greenA11: "color(display-p3 0.4338 0.8276 0.5733)",
    greenA12: "color(display-p3 0.7469 0.9381 0.8065)",
};
const grassDark = {
    grass1: "#0e1511",
    grass2: "#151a16",
    grass3: "#1b2a1e",
    grass4: "#1e3a25",
    grass5: "#25482e",
    grass6: "#2d5737",
    grass7: "#366740",
    grass8: "#3e7949",
    grass9: "#46a758",
    grass10: "#53b365",
    grass11: "#71d083",
    grass12: "#c2f0c2",
};
const grassDarkA = {
    grassA1: "#00de1205",
    grassA2: "#78f7910a",
    grassA3: "#70fe8c1b",
    grassA4: "#5dff852c",
    grassA5: "#68ff8f3b",
    grassA6: "#71ff934b",
    grassA7: "#77fd925d",
    grassA8: "#77fd9070",
    grassA9: "#65ff82a1",
    grassA10: "#72ff8dae",
    grassA11: "#89ff9fcd",
    grassA12: "#ceffceef",
};
const grassDarkP3 = {
    grass1: "color(display-p3 0.0617 0.0832 0.0667)",
    grass2: "color(display-p3 0.0858 0.1015 0.0877)",
    grass3: "color(display-p3 0.118 0.1631 0.1219)",
    grass4: "color(display-p3 0.1432 0.2241 0.1529)",
    grass5: "color(display-p3 0.1784 0.2788 0.1883)",
    grass6: "color(display-p3 0.2171 0.3364 0.2258)",
    grass7: "color(display-p3 0.2584 0.3996 0.2649)",
    grass8: "color(display-p3 0.3019 0.4697 0.3049)",
    grass9: "color(display-p3 0.3795 0.6466 0.3779)",
    grass10: "color(display-p3 0.4256 0.694 0.4262)",
    grass11: "color(display-p3 0.5349 0.8067 0.5421)",
    grass12: "color(display-p3 0.7968 0.9359 0.7757)",
};
const grassDarkP3A = {
    grassA1: "color(display-p3 0 0.9922 0.0706 / 0.017)",
    grassA2: "color(display-p3 0.5843 0.9961 0.5843 / 0.038)",
    grassA3: "color(display-p3 0.549 0.9922 0.5882 / 0.106)",
    grassA4: "color(display-p3 0.5333 0.9961 0.5804 / 0.169)",
    grassA5: "color(display-p3 0.5529 1 0.6039 / 0.227)",
    grassA6: "color(display-p3 0.5843 1 0.6235 / 0.29)",
    grassA7: "color(display-p3 0.6039 1 0.6275 / 0.358)",
    grassA8: "color(display-p3 0.6078 1 0.6196 / 0.433)",
    grassA9: "color(display-p3 0.5725 1 0.5686 / 0.622)",
    grassA10: "color(display-p3 0.6 0.9961 0.6 / 0.673)",
    grassA11: "color(display-p3 0.5349 0.8067 0.5421)",
    grassA12: "color(display-p3 0.7968 0.9359 0.7757)",
};
const brownDark = {
    brown1: "#12110f",
    brown2: "#1c1816",
    brown3: "#28211d",
    brown4: "#322922",
    brown5: "#3e3128",
    brown6: "#4d3c2f",
    brown7: "#614a39",
    brown8: "#7c5f46",
    brown9: "#ad7f58",
    brown10: "#b88c67",
    brown11: "#dbb594",
    brown12: "#f2e1ca",
};
const brownDarkA = {
    brownA1: "#91110002",
    brownA2: "#fba67c0c",
    brownA3: "#fcb58c19",
    brownA4: "#fbbb8a24",
    brownA5: "#fcb88931",
    brownA6: "#fdba8741",
    brownA7: "#ffbb8856",
    brownA8: "#ffbe8773",
    brownA9: "#feb87da8",
    brownA10: "#ffc18cb3",
    brownA11: "#fed1aad9",
    brownA12: "#feecd4f2",
};
const brownDarkP3 = {
    brown1: "color(display-p3 0.0707 0.0669 0.0594)",
    brown2: "color(display-p3 0.1073 0.0954 0.0873)",
    brown3: "color(display-p3 0.1513 0.1299 0.1148)",
    brown4: "color(display-p3 0.1914 0.1608 0.138)",
    brown5: "color(display-p3 0.2354 0.1943 0.1622)",
    brown6: "color(display-p3 0.2907 0.2366 0.1919)",
    brown7: "color(display-p3 0.3652 0.2946 0.2317)",
    brown8: "color(display-p3 0.4687 0.3774 0.2874)",
    brown9: "color(display-p3 0.651 0.5054 0.3677)",
    brown10: "color(display-p3 0.6966 0.5568 0.4231)",
    brown11: "color(display-p3 0.8349 0.7154 0.5971)",
    brown12: "color(display-p3 0.9376 0.8847 0.8021)",
};
const brownDarkP3A = {
    brownA1: "color(display-p3 0.8549 0.0706 0 / 0.005)",
    brownA2: "color(display-p3 0.9804 0.7059 0.5255 / 0.043)",
    brownA3: "color(display-p3 0.9961 0.7451 0.5765 / 0.093)",
    brownA4: "color(display-p3 1 0.7647 0.5922 / 0.135)",
    brownA5: "color(display-p3 1 0.7843 0.5882 / 0.181)",
    brownA6: "color(display-p3 1 0.7725 0.5922 / 0.24)",
    brownA7: "color(display-p3 0.9961 0.7765 0.5804 / 0.32)",
    brownA8: "color(display-p3 1 0.7804 0.5725 / 0.433)",
    brownA9: "color(display-p3 1 0.7686 0.549 / 0.627)",
    brownA10: "color(display-p3 1 0.7922 0.5961 / 0.677)",
    brownA11: "color(display-p3 0.8349 0.7154 0.5971)",
    brownA12: "color(display-p3 0.9376 0.8847 0.8021)",
};
const bronzeDark = {
    bronze1: "#141110",
    bronze2: "#1c1917",
    bronze3: "#262220",
    bronze4: "#302a27",
    bronze5: "#3b3330",
    bronze6: "#493e3a",
    bronze7: "#5a4c47",
    bronze8: "#6f5f58",
    bronze9: "#a18072",
    bronze10: "#ae8c7e",
    bronze11: "#d4b3a5",
    bronze12: "#ede0d9",
};
const bronzeDarkA = {
    bronzeA1: "#d1110004",
    bronzeA2: "#fbbc910c",
    bronzeA3: "#faceb817",
    bronzeA4: "#facdb622",
    bronzeA5: "#ffd2c12d",
    bronzeA6: "#ffd1c03c",
    bronzeA7: "#fdd0c04f",
    bronzeA8: "#ffd6c565",
    bronzeA9: "#fec7b09b",
    bronzeA10: "#fecab5a9",
    bronzeA11: "#ffd7c6d1",
    bronzeA12: "#fff1e9ec",
};
const bronzeDarkP3 = {
    bronze1: "color(display-p3 0.0764 0.0671 0.0633)",
    bronze2: "color(display-p3 0.106 0.0969 0.0926)",
    bronze3: "color(display-p3 0.1469 0.1323 0.1254)",
    bronze4: "color(display-p3 0.1854 0.1656 0.1562)",
    bronze5: "color(display-p3 0.2273 0.202 0.1896)",
    bronze6: "color(display-p3 0.2782 0.246 0.2301)",
    bronze7: "color(display-p3 0.3429 0.3019 0.2814)",
    bronze8: "color(display-p3 0.4261 0.3737 0.3472)",
    bronze9: "color(display-p3 0.6109 0.5069 0.4552)",
    bronze10: "color(display-p3 0.6599 0.5558 0.5039)",
    bronze11: "color(display-p3 0.8104 0.7068 0.6549)",
    bronze12: "color(display-p3 0.9206 0.8802 0.8544)",
};
const bronzeDarkP3A = {
    bronzeA1: "color(display-p3 0.9412 0.0667 0 / 0.009)",
    bronzeA2: "color(display-p3 0.9804 0.8 0.7059 / 0.043)",
    bronzeA3: "color(display-p3 0.9882 0.851 0.7608 / 0.085)",
    bronzeA4: "color(display-p3 0.9961 0.8392 0.7804 / 0.127)",
    bronzeA5: "color(display-p3 0.9961 0.8627 0.7725 / 0.173)",
    bronzeA6: "color(display-p3 1 0.8627 0.7961 / 0.227)",
    bronzeA7: "color(display-p3 1 0.8667 0.8 / 0.295)",
    bronzeA8: "color(display-p3 1 0.8588 0.8 / 0.387)",
    bronzeA9: "color(display-p3 1 0.8196 0.7333 / 0.585)",
    bronzeA10: "color(display-p3 1 0.8392 0.7529 / 0.635)",
    bronzeA11: "color(display-p3 0.8104 0.7068 0.6549)",
    bronzeA12: "color(display-p3 0.9206 0.8802 0.8544)",
};
const goldDark = {
    gold1: "#121211",
    gold2: "#1b1a17",
    gold3: "#24231f",
    gold4: "#2d2b26",
    gold5: "#38352e",
    gold6: "#444039",
    gold7: "#544f46",
    gold8: "#696256",
    gold9: "#978365",
    gold10: "#a39073",
    gold11: "#cbb99f",
    gold12: "#e8e2d9",
};
const goldDarkA = {
    goldA1: "#91911102",
    goldA2: "#f9e29d0b",
    goldA3: "#f8ecbb15",
    goldA4: "#ffeec41e",
    goldA5: "#feecc22a",
    goldA6: "#feebcb37",
    goldA7: "#ffedcd48",
    goldA8: "#fdeaca5f",
    goldA9: "#ffdba690",
    goldA10: "#fedfb09d",
    goldA11: "#fee7c6c8",
    goldA12: "#fef7ede7",
};
const goldDarkP3 = {
    gold1: "color(display-p3 0.0706 0.0706 0.067)",
    gold2: "color(display-p3 0.104 0.1005 0.0904)",
    gold3: "color(display-p3 0.1409 0.1357 0.1215)",
    gold4: "color(display-p3 0.1768 0.1696 0.1516)",
    gold5: "color(display-p3 0.2166 0.207 0.1848)",
    gold6: "color(display-p3 0.2652 0.2525 0.2252)",
    gold7: "color(display-p3 0.3274 0.3101 0.2767)",
    gold8: "color(display-p3 0.4074 0.3835 0.3424)",
    gold9: "color(display-p3 0.5792 0.5166 0.4103)",
    gold10: "color(display-p3 0.6275 0.5661 0.4626)",
    gold11: "color(display-p3 0.7841 0.728 0.6351)",
    gold12: "color(display-p3 0.9057 0.8871 0.8546)",
};
const goldDarkP3A = {
    goldA1: "color(display-p3 0.8549 0.8549 0.0706 / 0.005)",
    goldA2: "color(display-p3 0.9804 0.8902 0.6157 / 0.043)",
    goldA3: "color(display-p3 1 0.949 0.7529 / 0.08)",
    goldA4: "color(display-p3 1 0.9333 0.8 / 0.118)",
    goldA5: "color(display-p3 1 0.949 0.8039 / 0.16)",
    goldA6: "color(display-p3 1 0.9255 0.8 / 0.215)",
    goldA7: "color(display-p3 1 0.9451 0.8314 / 0.278)",
    goldA8: "color(display-p3 1 0.9373 0.8196 / 0.366)",
    goldA9: "color(display-p3 0.9961 0.8824 0.6902 / 0.551)",
    goldA10: "color(display-p3 1 0.8941 0.7255 / 0.601)",
    goldA11: "color(display-p3 0.7841 0.728 0.6351)",
    goldA12: "color(display-p3 0.9057 0.8871 0.8546)",
};
const skyDark = {
    sky1: "#0d141f",
    sky2: "#121a26",
    sky3: "#12283e",
    sky4: "#123553",
    sky5: "#154366",
    sky6: "#1b537a",
    sky7: "#1f6691",
    sky8: "#197cae",
    sky9: "#7ce2fe",
    sky10: "#a8eeff",
    sky11: "#75c7f0",
    sky12: "#c2f3ff",
};
const skyDarkA = {
    skyA1: "#0044ff0f",
    skyA2: "#1d75fa17",
    skyA3: "#1789fc31",
    skyA4: "#1593ff47",
    skyA5: "#1d9cfd5c",
    skyA6: "#28a6fe71",
    skyA7: "#2baefe8a",
    skyA8: "#1db2fea9",
    skyA9: "#7ce3fffe",
    skyA10: "#a8eeff",
    skyA11: "#7cd3ffef",
    skyA12: "#c2f3ff",
};
const skyDarkP3 = {
    sky1: "color(display-p3 0.0561 0.0784 0.1164)",
    sky2: "color(display-p3 0.078 0.1027 0.1447)",
    sky3: "color(display-p3 0.0907 0.1546 0.2378)",
    sky4: "color(display-p3 0.1075 0.206 0.3164)",
    sky5: "color(display-p3 0.1354 0.2604 0.3882)",
    sky6: "color(display-p3 0.1707 0.3221 0.4648)",
    sky7: "color(display-p3 0.2056 0.3947 0.5555)",
    sky8: "color(display-p3 0.2318 0.4803 0.6655)",
    sky9: "color(display-p3 0.5845 0.8766 0.9826)",
    sky10: "color(display-p3 0.7184 0.9246 0.9907)",
    sky11: "color(display-p3 0.5355 0.772 0.9239)",
    sky12: "color(display-p3 0.7994 0.9474 0.9932)",
};
const skyDarkP3A = {
    skyA1: "color(display-p3 0 0.2824 0.9961 / 0.055)",
    skyA2: "color(display-p3 0.2039 0.4824 0.9882 / 0.085)",
    skyA3: "color(display-p3 0.1961 0.5333 1 / 0.185)",
    skyA4: "color(display-p3 0.2157 0.5922 1 / 0.269)",
    skyA5: "color(display-p3 0.2745 0.6275 1 / 0.345)",
    skyA6: "color(display-p3 0.3137 0.6588 1 / 0.429)",
    skyA7: "color(display-p3 0.3294 0.6941 1 / 0.526)",
    skyA8: "color(display-p3 0.3255 0.7098 1 / 0.643)",
    skyA9: "color(display-p3 0.5922 0.8941 1 / 0.984)",
    skyA10: "color(display-p3 0.7216 0.9333 1 / 0.992)",
    skyA11: "color(display-p3 0.5355 0.772 0.9239)",
    skyA12: "color(display-p3 0.7994 0.9474 0.9932)",
};
const mintDark = {
    mint1: "#0e1515",
    mint2: "#131c1c",
    mint3: "#092c2b",
    mint4: "#003b38",
    mint5: "#044843",
    mint6: "#14574f",
    mint7: "#20695f",
    mint8: "#277f70",
    mint9: "#86ead4",
    mint10: "#a8f5e5",
    mint11: "#58d5ba",
    mint12: "#c4f5e1",
};
const mintDarkA = {
    mintA1: "#00dede05",
    mintA2: "#3cfbfb0c",
    mintA3: "#00fff61d",
    mintA4: "#00ffee2d",
    mintA5: "#00ffea3b",
    mintA6: "#1cffe44b",
    mintA7: "#39fde25f",
    mintA8: "#41ffdf76",
    mintA9: "#92ffe7e9",
    mintA10: "#aefeedf5",
    mintA11: "#67ffded2",
    mintA12: "#cbfee9f5",
};
const mintDarkP3 = {
    mint1: "color(display-p3 0.0592 0.0821 0.0807)",
    mint2: "color(display-p3 0.0812 0.1094 0.1076)",
    mint3: "color(display-p3 0.0772 0.1704 0.1676)",
    mint4: "color(display-p3 0.082 0.2292 0.2173)",
    mint5: "color(display-p3 0.1155 0.2791 0.2616)",
    mint6: "color(display-p3 0.1627 0.335 0.3111)",
    mint7: "color(display-p3 0.2116 0.4047 0.3712)",
    mint8: "color(display-p3 0.2581 0.4903 0.4412)",
    mint9: "color(display-p3 0.6198 0.908 0.834)",
    mint10: "color(display-p3 0.7245 0.9539 0.8978)",
    mint11: "color(display-p3 0.4824 0.8254 0.7334)",
    mint12: "color(display-p3 0.8072 0.9552 0.8866)",
};
const mintDarkP3A = {
    mintA1: "color(display-p3 0 0.9922 0.9922 / 0.017)",
    mintA2: "color(display-p3 0.4039 0.9882 0.902 / 0.047)",
    mintA3: "color(display-p3 0.1765 0.9961 0.9961 / 0.11)",
    mintA4: "color(display-p3 0.1608 0.9961 0.9294 / 0.173)",
    mintA5: "color(display-p3 0.2745 1 0.9333 / 0.227)",
    mintA6: "color(display-p3 0.3961 1 0.9176 / 0.286)",
    mintA7: "color(display-p3 0.4706 1 0.9137 / 0.362)",
    mintA8: "color(display-p3 0.4902 1 0.898 / 0.454)",
    mintA9: "color(display-p3 0.6784 0.9961 0.9137 / 0.904)",
    mintA10: "color(display-p3 0.7608 1 0.9412 / 0.95)",
    mintA11: "color(display-p3 0.4824 0.8254 0.7334)",
    mintA12: "color(display-p3 0.8072 0.9552 0.8866)",
};
const limeDark = {
    lime1: "#111304",
    lime2: "#161913",
    lime3: "#202919",
    lime4: "#2a371f",
    lime5: "#334424",
    lime6: "#3e522b",
    lime7: "#496231",
    lime8: "#577538",
    lime9: "#bdee63",
    lime10: "#d4ff70",
    lime11: "#bde56c",
    lime12: "#e3f7ba",
};
const limeDarkA = {
    limeA1: "#11bb0003",
    limeA2: "#9ff44a09",
    limeA3: "#a5fd601a",
    limeA4: "#adfe6929",
    limeA5: "#affe6a37",
    limeA6: "#b5fe7046",
    limeA7: "#b6ff6f57",
    limeA8: "#b6fd6d6c",
    limeA9: "#caff69ed",
    limeA10: "#d4ff70",
    limeA11: "#d1fe77e4",
    limeA12: "#e9febff7",
};
const limeDarkP3 = {
    lime1: "color(display-p3 0.0681 0.0743 0.0216)",
    lime2: "color(display-p3 0.0884 0.0985 0.0754)",
    lime3: "color(display-p3 0.1329 0.1591 0.1043)",
    lime4: "color(display-p3 0.1738 0.2132 0.1302)",
    lime5: "color(display-p3 0.2143 0.2651 0.1566)",
    lime6: "color(display-p3 0.2579 0.3204 0.1849)",
    lime7: "color(display-p3 0.3075 0.3831 0.216)",
    lime8: "color(display-p3 0.3647 0.4557 0.2502)",
    lime9: "color(display-p3 0.7799 0.9278 0.4663)",
    lime10: "color(display-p3 0.865 0.995 0.5187)",
    lime11: "color(display-p3 0.7707 0.8933 0.4855)",
    lime12: "color(display-p3 0.9048 0.9662 0.7532)",
};
const limeDarkP3A = {
    limeA1: "color(display-p3 0.0667 0.9412 0 / 0.009)",
    limeA2: "color(display-p3 0.7608 0.9922 0.298 / 0.034)",
    limeA3: "color(display-p3 0.7294 1 0.4588 / 0.101)",
    limeA4: "color(display-p3 0.749 1 0.4706 / 0.156)",
    limeA5: "color(display-p3 0.7608 1 0.4863 / 0.215)",
    limeA6: "color(display-p3 0.7686 1 0.498 / 0.274)",
    limeA7: "color(display-p3 0.7686 1 0.5059 / 0.341)",
    limeA8: "color(display-p3 0.7843 1 0.5098 / 0.416)",
    limeA9: "color(display-p3 0.8392 1 0.502 / 0.925)",
    limeA10: "color(display-p3 0.8706 1 0.5216 / 0.996)",
    limeA11: "color(display-p3 0.7707 0.8933 0.4855)",
    limeA12: "color(display-p3 0.9048 0.9662 0.7532)",
};
const yellowDark = {
    yellow1: "#14120b",
    yellow2: "#1a1812",
    yellow3: "#2d2305",
    yellow4: "#382a00",
    yellow5: "#443400",
    yellow6: "#534103",
    yellow7: "#675318",
    yellow8: "#836a21",
    yellow9: "#ffe629",
    yellow10: "#ffff57",
    yellow11: "#f5e147",
    yellow12: "#f6eeb4",
};
const yellowDarkA = {
    yellowA1: "#d1510004",
    yellowA2: "#f7c42b0a",
    yellowA3: "#ffaa001e",
    yellowA4: "#fea9002a",
    yellowA5: "#feb40037",
    yellowA6: "#ffbe0047",
    yellowA7: "#fdc6255d",
    yellowA8: "#fdca327b",
    yellowA9: "#ffe629",
    yellowA10: "#ffff57",
    yellowA11: "#fee949f5",
    yellowA12: "#fef6baf6",
};
const yellowDarkP3 = {
    yellow1: "color(display-p3 0.0776 0.0693 0.0474)",
    yellow2: "color(display-p3 0.1014 0.0932 0.0715)",
    yellow3: "color(display-p3 0.1684 0.1373 0.0388)",
    yellow4: "color(display-p3 0.213 0.1663 0)",
    yellow5: "color(display-p3 0.2595 0.206 0)",
    yellow6: "color(display-p3 0.3135 0.2587 0.0717)",
    yellow7: "color(display-p3 0.3906 0.3294 0.1371)",
    yellow8: "color(display-p3 0.4966 0.4196 0.1824)",
    yellow9: "color(display-p3 1 0.92 0.22)",
    yellow10: "color(display-p3 1 1 0.4561)",
    yellow11: "color(display-p3 0.9485 0.885 0.392)",
    yellow12: "color(display-p3 0.9592 0.9344 0.7309)",
};
const yellowDarkP3A = {
    yellowA1: "color(display-p3 0.9725 0.3686 0 / 0.013)",
    yellowA2: "color(display-p3 0.9961 0.7922 0.1725 / 0.038)",
    yellowA3: "color(display-p3 0.9961 0.7098 0 / 0.11)",
    yellowA4: "color(display-p3 1 0.698 0 / 0.156)",
    yellowA5: "color(display-p3 1 0.7529 0 / 0.206)",
    yellowA6: "color(display-p3 1 0.7922 0.0824 / 0.265)",
    yellowA7: "color(display-p3 1 0.8196 0.2706 / 0.349)",
    yellowA8: "color(display-p3 1 0.8314 0.3216 / 0.463)",
    yellowA9: "color(display-p3 1 0.9216 0.2196)",
    yellowA10: "color(display-p3 1 1 0.4549)",
    yellowA11: "color(display-p3 0.9485 0.885 0.392)",
    yellowA12: "color(display-p3 0.9592 0.9344 0.7309)",
};
const amberDark = {
    amber1: "#16120c",
    amber2: "#1c1812",
    amber3: "#302008",
    amber4: "#3e2700",
    amber5: "#4c3000",
    amber6: "#5b3d06",
    amber7: "#704f1a",
    amber8: "#8f6424",
    amber9: "#ffc53d",
    amber10: "#ffd60a",
    amber11: "#ffca16",
    amber12: "#ffe7b3",
};
const amberDarkA = {
    amberA1: "#e63c0006",
    amberA2: "#fba6270c",
    amberA3: "#fa820022",
    amberA4: "#fc840031",
    amberA5: "#fd8d0040",
    amberA6: "#fd9d0050",
    amberA7: "#ffac2866",
    amberA8: "#ffae3587",
    amberA9: "#ffc53d",
    amberA10: "#ffd60a",
    amberA11: "#ffca16",
    amberA12: "#ffe7b3",
};
const amberDarkP3 = {
    amber1: "color(display-p3 0.082 0.0702 0.0501)",
    amber2: "color(display-p3 0.1056 0.0941 0.0745)",
    amber3: "color(display-p3 0.1777 0.1279 0.0493)",
    amber4: "color(display-p3 0.2356 0.1567 0)",
    amber5: "color(display-p3 0.2866 0.1935 0)",
    amber6: "color(display-p3 0.3409 0.2458 0.0767)",
    amber7: "color(display-p3 0.4202 0.3146 0.1417)",
    amber8: "color(display-p3 0.5349 0.399 0.1888)",
    amber9: "color(display-p3 1 0.77 0.26)",
    amber10: "color(display-p3 1 0.87 0.15)",
    amber11: "color(display-p3 1 0.8 0.29)",
    amber12: "color(display-p3 0.9842 0.9092 0.7256)",
};
const amberDarkP3A = {
    amberA1: "color(display-p3 0.9922 0.298 0 / 0.017)",
    amberA2: "color(display-p3 0.9804 0.7059 0.251 / 0.043)",
    amberA3: "color(display-p3 1 0.6 0 / 0.118)",
    amberA4: "color(display-p3 1 0.5686 0 / 0.181)",
    amberA5: "color(display-p3 1 0.6 0 / 0.236)",
    amberA6: "color(display-p3 1 0.6784 0.1098 / 0.295)",
    amberA7: "color(display-p3 0.9961 0.7176 0.2627 / 0.379)",
    amberA8: "color(display-p3 0.9961 0.7294 0.3059 / 0.5)",
    amberA9: "color(display-p3 1 0.7686 0.2588)",
    amberA10: "color(display-p3 1 0.8706 0.149)",
    amberA11: "color(display-p3 1 0.8 0.29)",
    amberA12: "color(display-p3 0.9842 0.9092 0.7256)",
};
const orangeDark = {
    orange1: "#17120e",
    orange2: "#1c1712",
    orange3: "#331e0b",
    orange4: "#462000",
    orange5: "#562800",
    orange6: "#66350e",
    orange7: "#7e451e",
    orange8: "#a35829",
    orange9: "#f76b15",
    orange10: "#ff801f",
    orange11: "#ffa057",
    orange12: "#ffe0c2",
};
const orangeDarkA = {
    orangeA1: "#ec360007",
    orangeA2: "#fb91270c",
    orangeA3: "#fb6a0025",
    orangeA4: "#ff550039",
    orangeA5: "#ff61004a",
    orangeA6: "#fd75095c",
    orangeA7: "#ff832e75",
    orangeA8: "#fe84389d",
    orangeA9: "#fe6d15f7",
    orangeA10: "#ff801f",
    orangeA11: "#ffa057",
    orangeA12: "#ffe0c2",
};
const orangeDarkP3 = {
    orange1: "color(display-p3 0.0877 0.0702 0.0567)",
    orange2: "color(display-p3 0.1071 0.0894 0.0717)",
    orange3: "color(display-p3 0.1892 0.1205 0.0562)",
    orange4: "color(display-p3 0.2645 0.1289 0)",
    orange5: "color(display-p3 0.3155 0.1658 0.0225)",
    orange6: "color(display-p3 0.3758 0.2175 0.092)",
    orange7: "color(display-p3 0.4649 0.2826 0.1493)",
    orange8: "color(display-p3 0.6006 0.359 0.2006)",
    orange9: "color(display-p3 0.9 0.45 0.2)",
    orange10: "color(display-p3 0.98 0.51 0.23)",
    orange11: "color(display-p3 1 0.63 0.38)",
    orange12: "color(display-p3 0.9799 0.8828 0.775)",
};
const orangeDarkP3A = {
    orangeA1: "color(display-p3 0.9608 0.2471 0 / 0.022)",
    orangeA2: "color(display-p3 0.9804 0.6157 0.1608 / 0.043)",
    orangeA3: "color(display-p3 0.9961 0.4863 0 / 0.131)",
    orangeA4: "color(display-p3 0.9961 0.3647 0 / 0.211)",
    orangeA5: "color(display-p3 1 0.4392 0 / 0.265)",
    orangeA6: "color(display-p3 1 0.5176 0.1412 / 0.332)",
    orangeA7: "color(display-p3 1 0.5686 0.2588 / 0.429)",
    orangeA8: "color(display-p3 1 0.5843 0.302 / 0.572)",
    orangeA9: "color(display-p3 1 0.4941 0.2157 / 0.895)",
    orangeA10: "color(display-p3 1 0.5216 0.2353 / 0.979)",
    orangeA11: "color(display-p3 1 0.63 0.38)",
    orangeA12: "color(display-p3 0.9799 0.8828 0.775)",
};

const gray = {
    gray1: "#fcfcfc",
    gray2: "#f9f9f9",
    gray3: "#f0f0f0",
    gray4: "#e8e8e8",
    gray5: "#e0e0e0",
    gray6: "#d9d9d9",
    gray7: "#cecece",
    gray8: "#bbbbbb",
    gray9: "#8d8d8d",
    gray10: "#838383",
    gray11: "#646464",
    gray12: "#202020",
};
const grayA = {
    grayA1: "#00000003",
    grayA2: "#00000006",
    grayA3: "#0000000f",
    grayA4: "#00000017",
    grayA5: "#0000001f",
    grayA6: "#00000026",
    grayA7: "#00000031",
    grayA8: "#00000044",
    grayA9: "#00000072",
    grayA10: "#0000007c",
    grayA11: "#0000009b",
    grayA12: "#000000df",
};
const grayP3 = {
    gray1: "color(display-p3 0.9882 0.9882 0.9882)",
    gray2: "color(display-p3 0.975 0.975 0.975)",
    gray3: "color(display-p3 0.9394 0.9394 0.9394)",
    gray4: "color(display-p3 0.9082 0.9082 0.9082)",
    gray5: "color(display-p3 0.8796 0.8796 0.8796)",
    gray6: "color(display-p3 0.8494 0.8494 0.8494)",
    gray7: "color(display-p3 0.8075 0.8075 0.8075)",
    gray8: "color(display-p3 0.732 0.732 0.732)",
    gray9: "color(display-p3 0.5529 0.5529 0.5529)",
    gray10: "color(display-p3 0.5118 0.5118 0.5118)",
    gray11: "color(display-p3 0.3922 0.3922 0.3922)",
    gray12: "color(display-p3 0.1255 0.1255 0.1255)",
};
const grayP3A = {
    grayA1: "color(display-p3 0 0 0 / 0.0118)",
    grayA2: "color(display-p3 0 0 0 / 0.0235)",
    grayA3: "color(display-p3 0 0 0 / 0.0588)",
    grayA4: "color(display-p3 0 0 0 / 0.0902)",
    grayA5: "color(display-p3 0 0 0 / 0.1216)",
    grayA6: "color(display-p3 0 0 0 / 0.149)",
    grayA7: "color(display-p3 0 0 0 / 0.1922)",
    grayA8: "color(display-p3 0 0 0 / 0.2667)",
    grayA9: "color(display-p3 0 0 0 / 0.4471)",
    grayA10: "color(display-p3 0 0 0 / 0.4863)",
    grayA11: "color(display-p3 0 0 0 / 0.6078)",
    grayA12: "color(display-p3 0 0 0 / 0.8745)",
};
const mauve = {
    mauve1: "#fdfcfd",
    mauve2: "#faf9fb",
    mauve3: "#f2eff3",
    mauve4: "#eae7ec",
    mauve5: "#e3dfe6",
    mauve6: "#dbd8e0",
    mauve7: "#d0cdd7",
    mauve8: "#bcbac7",
    mauve9: "#8e8c99",
    mauve10: "#84828e",
    mauve11: "#65636d",
    mauve12: "#211f26",
};
const mauveA = {
    mauveA1: "#55005503",
    mauveA2: "#2b005506",
    mauveA3: "#30004010",
    mauveA4: "#20003618",
    mauveA5: "#20003820",
    mauveA6: "#14003527",
    mauveA7: "#10003332",
    mauveA8: "#08003145",
    mauveA9: "#05001d73",
    mauveA10: "#0500197d",
    mauveA11: "#0400119c",
    mauveA12: "#020008e0",
};
const mauveP3 = {
    mauve1: "color(display-p3 0.9915 0.9884 0.9919)",
    mauve2: "color(display-p3 0.9804 0.9756 0.9839)",
    mauve3: "color(display-p3 0.9455 0.9382 0.9524)",
    mauve4: "color(display-p3 0.9147 0.9058 0.9252)",
    mauve5: "color(display-p3 0.8863 0.8765 0.9008)",
    mauve6: "color(display-p3 0.8563 0.846 0.8754)",
    mauve7: "color(display-p3 0.8141 0.8039 0.8404)",
    mauve8: "color(display-p3 0.7354 0.7284 0.7775)",
    mauve9: "color(display-p3 0.5555 0.5493 0.5958)",
    mauve10: "color(display-p3 0.5144 0.5082 0.5519)",
    mauve11: "color(display-p3 0.3947 0.3885 0.4242)",
    mauve12: "color(display-p3 0.1281 0.1218 0.1469)",
};
const mauveP3A = {
    mauveA1: "color(display-p3 0.349 0.0235 0.349 / 0.012)",
    mauveA2: "color(display-p3 0.1843 0.0235 0.349 / 0.024)",
    mauveA3: "color(display-p3 0.1294 0.0078 0.2549 / 0.063)",
    mauveA4: "color(display-p3 0.0941 0.0118 0.2157 / 0.095)",
    mauveA5: "color(display-p3 0.0706 0.0039 0.2 / 0.122)",
    mauveA6: "color(display-p3 0.0549 0.0039 0.1804 / 0.153)",
    mauveA7: "color(display-p3 0.0667 0.0078 0.1843 / 0.197)",
    mauveA8: "color(display-p3 0.0314 0.0039 0.1765 / 0.271)",
    mauveA9: "color(display-p3 0.0196 0.0039 0.1059 / 0.451)",
    mauveA10: "color(display-p3 0.0118 0.0039 0.0902 / 0.491)",
    mauveA11: "color(display-p3 0.0157 0 0.0588 / 0.612)",
    mauveA12: "color(display-p3 0.0078 0 0.0275 / 0.879)",
};
const slate = {
    slate1: "#fcfcfd",
    slate2: "#f9f9fb",
    slate3: "#f0f0f3",
    slate4: "#e8e8ec",
    slate5: "#e0e1e6",
    slate6: "#d9d9e0",
    slate7: "#cdced6",
    slate8: "#b9bbc6",
    slate9: "#8b8d98",
    slate10: "#80838d",
    slate11: "#60646c",
    slate12: "#1c2024",
};
const slateA = {
    slateA1: "#00005503",
    slateA2: "#00005506",
    slateA3: "#0000330f",
    slateA4: "#00002d17",
    slateA5: "#0009321f",
    slateA6: "#00002f26",
    slateA7: "#00062e32",
    slateA8: "#00083046",
    slateA9: "#00051d74",
    slateA10: "#00071b7f",
    slateA11: "#0007149f",
    slateA12: "#000509e3",
};
const slateP3 = {
    slate1: "color(display-p3 0.9882 0.9882 0.9918)",
    slate2: "color(display-p3 0.9765 0.9765 0.9836)",
    slate3: "color(display-p3 0.9403 0.9405 0.9526)",
    slate4: "color(display-p3 0.9084 0.909 0.9254)",
    slate5: "color(display-p3 0.8798 0.881 0.9012)",
    slate6: "color(display-p3 0.85 0.8519 0.876)",
    slate7: "color(display-p3 0.8052 0.8084 0.8382)",
    slate8: "color(display-p3 0.7269 0.7331 0.7726)",
    slate9: "color(display-p3 0.5465 0.5527 0.5923)",
    slate10: "color(display-p3 0.5034 0.5116 0.5485)",
    slate11: "color(display-p3 0.3793 0.3916 0.4206)",
    slate12: "color(display-p3 0.1127 0.125 0.1396)",
};
const slateP3A = {
    slateA1: "color(display-p3 0.0235 0.0235 0.349 / 0.012)",
    slateA2: "color(display-p3 0.0235 0.0235 0.349 / 0.024)",
    slateA3: "color(display-p3 0.0039 0.0039 0.2039 / 0.059)",
    slateA4: "color(display-p3 0.0118 0.0118 0.1843 / 0.091)",
    slateA5: "color(display-p3 0.0039 0.0392 0.2 / 0.122)",
    slateA6: "color(display-p3 0.0078 0.0078 0.1647 / 0.15)",
    slateA7: "color(display-p3 0.0078 0.0275 0.1843 / 0.197)",
    slateA8: "color(display-p3 0.0039 0.0314 0.1765 / 0.275)",
    slateA9: "color(display-p3 0.0039 0.0196 0.1059 / 0.455)",
    slateA10: "color(display-p3 0.0039 0.0196 0.098 / 0.499)",
    slateA11: "color(display-p3 0 0.0196 0.0627 / 0.62)",
    slateA12: "color(display-p3 0 0.0118 0.0314 / 0.887)",
};
const sage = {
    sage1: "#fbfdfc",
    sage2: "#f7f9f8",
    sage3: "#eef1f0",
    sage4: "#e6e9e8",
    sage5: "#dfe2e0",
    sage6: "#d7dad9",
    sage7: "#cbcfcd",
    sage8: "#b8bcba",
    sage9: "#868e8b",
    sage10: "#7c8481",
    sage11: "#5f6563",
    sage12: "#1a211e",
};
const sageA = {
    sageA1: "#00804004",
    sageA2: "#00402008",
    sageA3: "#002d1e11",
    sageA4: "#001f1519",
    sageA5: "#00180820",
    sageA6: "#00140d28",
    sageA7: "#00140a34",
    sageA8: "#000f0847",
    sageA9: "#00110b79",
    sageA10: "#00100a83",
    sageA11: "#000a07a0",
    sageA12: "#000805e5",
};
const sageP3 = {
    sage1: "color(display-p3 0.9857 0.9919 0.9885)",
    sage2: "color(display-p3 0.97 0.9773 0.974)",
    sage3: "color(display-p3 0.9355 0.9435 0.9398)",
    sage4: "color(display-p3 0.9044 0.9132 0.9091)",
    sage5: "color(display-p3 0.8753 0.8848 0.8804)",
    sage6: "color(display-p3 0.8439 0.8541 0.8494)",
    sage7: "color(display-p3 0.7997 0.8108 0.8056)",
    sage8: "color(display-p3 0.7248 0.7376 0.7317)",
    sage9: "color(display-p3 0.5312 0.5559 0.5456)",
    sage10: "color(display-p3 0.4917 0.5148 0.5055)",
    sage11: "color(display-p3 0.3769 0.3953 0.3885)",
    sage12: "color(display-p3 0.1073 0.1286 0.1183)",
};
const sageP3A = {
    sageA1: "color(display-p3 0.0235 0.5137 0.2667 / 0.016)",
    sageA2: "color(display-p3 0.0196 0.2667 0.1451 / 0.032)",
    sageA3: "color(display-p3 0.0078 0.1294 0.0667 / 0.063)",
    sageA4: "color(display-p3 0.0118 0.0941 0.051 / 0.095)",
    sageA5: "color(display-p3 0.0078 0.098 0.0667 / 0.126)",
    sageA6: "color(display-p3 0.0039 0.0784 0.051 / 0.157)",
    sageA7: "color(display-p3 0 0.0588 0.0196 / 0.2)",
    sageA8: "color(display-p3 0.0039 0.0471 0.0314 / 0.275)",
    sageA9: "color(display-p3 0.0039 0.0588 0.0353 / 0.471)",
    sageA10: "color(display-p3 0 0.0471 0.0314 / 0.51)",
    sageA11: "color(display-p3 0 0.0314 0.0196 / 0.624)",
    sageA12: "color(display-p3 0 0.0275 0.0118 / 0.895)",
};
const olive = {
    olive1: "#fcfdfc",
    olive2: "#f8faf8",
    olive3: "#eff1ef",
    olive4: "#e7e9e7",
    olive5: "#dfe2df",
    olive6: "#d7dad7",
    olive7: "#cccfcc",
    olive8: "#b9bcb8",
    olive9: "#898e87",
    olive10: "#7f847d",
    olive11: "#60655f",
    olive12: "#1d211c",
};
const oliveA = {
    oliveA1: "#00550003",
    oliveA2: "#00490007",
    oliveA3: "#00200010",
    oliveA4: "#00160018",
    oliveA5: "#00180020",
    oliveA6: "#00140028",
    oliveA7: "#000f0033",
    oliveA8: "#040f0047",
    oliveA9: "#050f0078",
    oliveA10: "#040e0082",
    oliveA11: "#020a00a0",
    oliveA12: "#010600e3",
};
const oliveP3 = {
    olive1: "color(display-p3 0.9889 0.992 0.9885)",
    olive2: "color(display-p3 0.9745 0.9799 0.9725)",
    olive3: "color(display-p3 0.9392 0.9453 0.9369)",
    olive4: "color(display-p3 0.9074 0.9142 0.9049)",
    olive5: "color(display-p3 0.8777 0.8852 0.875)",
    olive6: "color(display-p3 0.8464 0.8545 0.8434)",
    olive7: "color(display-p3 0.8034 0.8124 0.8001)",
    olive8: "color(display-p3 0.727 0.7377 0.7231)",
    olive9: "color(display-p3 0.5408 0.5562 0.5316)",
    olive10: "color(display-p3 0.4997 0.5151 0.4914)",
    olive11: "color(display-p3 0.38 0.3954 0.3744)",
    olive12: "color(display-p3 0.1167 0.1289 0.1114)",
};
const oliveP3A = {
    oliveA1: "color(display-p3 0.0235 0.349 0.0235 / 0.012)",
    oliveA2: "color(display-p3 0.0235 0.302 0.0235 / 0.028)",
    oliveA3: "color(display-p3 0.0078 0.1294 0.0078 / 0.063)",
    oliveA4: "color(display-p3 0.0118 0.0941 0.0118 / 0.095)",
    oliveA5: "color(display-p3 0.0353 0.098 0.0078 / 0.126)",
    oliveA6: "color(display-p3 0.0275 0.0784 0.0039 / 0.157)",
    oliveA7: "color(display-p3 0.0196 0.0588 0 / 0.2)",
    oliveA8: "color(display-p3 0.0196 0.0588 0.0039 / 0.279)",
    oliveA9: "color(display-p3 0.0196 0.051 0.0039 / 0.467)",
    oliveA10: "color(display-p3 0.0157 0.0471 0 / 0.51)",
    oliveA11: "color(display-p3 0.0118 0.0392 0 / 0.628)",
    oliveA12: "color(display-p3 0.0078 0.0235 0 / 0.891)",
};
const sand = {
    sand1: "#fdfdfc",
    sand2: "#f9f9f8",
    sand3: "#f1f0ef",
    sand4: "#e9e8e6",
    sand5: "#e2e1de",
    sand6: "#dad9d6",
    sand7: "#cfceca",
    sand8: "#bcbbb5",
    sand9: "#8d8d86",
    sand10: "#82827c",
    sand11: "#63635e",
    sand12: "#21201c",
};
const sandA = {
    sandA1: "#55550003",
    sandA2: "#25250007",
    sandA3: "#20100010",
    sandA4: "#1f150019",
    sandA5: "#1f180021",
    sandA6: "#19130029",
    sandA7: "#19140035",
    sandA8: "#1915014a",
    sandA9: "#0f0f0079",
    sandA10: "#0c0c0083",
    sandA11: "#080800a1",
    sandA12: "#060500e3",
};
const sandP3 = {
    sand1: "color(display-p3 0.9922 0.9922 0.9886)",
    sand2: "color(display-p3 0.9771 0.9765 0.9731)",
    sand3: "color(display-p3 0.943 0.942 0.936)",
    sand4: "color(display-p3 0.913 0.9116 0.9033)",
    sand5: "color(display-p3 0.8848 0.883 0.8727)",
    sand6: "color(display-p3 0.8542 0.852 0.8395)",
    sand7: "color(display-p3 0.8126 0.8099 0.7943)",
    sand8: "color(display-p3 0.7379 0.7343 0.7134)",
    sand9: "color(display-p3 0.5529 0.5529 0.528)",
    sand10: "color(display-p3 0.5108 0.5108 0.4877)",
    sand11: "color(display-p3 0.3882 0.3882 0.3704)",
    sand12: "color(display-p3 0.1287 0.1256 0.1114)",
};
const sandP3A = {
    sandA1: "color(display-p3 0.349 0.349 0.0235 / 0.012)",
    sandA2: "color(display-p3 0.1608 0.1608 0.0235 / 0.028)",
    sandA3: "color(display-p3 0.0667 0.0667 0.0078 / 0.063)",
    sandA4: "color(display-p3 0.1294 0.0902 0.0118 / 0.099)",
    sandA5: "color(display-p3 0.098 0.0667 0.0078 / 0.126)",
    sandA6: "color(display-p3 0.102 0.0745 0.0039 / 0.161)",
    sandA7: "color(display-p3 0.0784 0.0784 0.0039 / 0.204)",
    sandA8: "color(display-p3 0.0863 0.0745 0.0039 / 0.287)",
    sandA9: "color(display-p3 0.051 0.051 0.0039 / 0.471)",
    sandA10: "color(display-p3 0.0471 0.0471 0 / 0.514)",
    sandA11: "color(display-p3 0.0314 0.0314 0 / 0.632)",
    sandA12: "color(display-p3 0.0235 0.0196 0 / 0.891)",
};
const tomato = {
    tomato1: "#fffcfc",
    tomato2: "#fff8f7",
    tomato3: "#feebe7",
    tomato4: "#ffdcd3",
    tomato5: "#ffcdc2",
    tomato6: "#fdbdaf",
    tomato7: "#f5a898",
    tomato8: "#ec8e7b",
    tomato9: "#e54d2e",
    tomato10: "#dd4425",
    tomato11: "#d13415",
    tomato12: "#5c271f",
};
const tomatoA = {
    tomatoA1: "#ff000003",
    tomatoA2: "#ff200008",
    tomatoA3: "#f52b0018",
    tomatoA4: "#ff35002c",
    tomatoA5: "#ff2e003d",
    tomatoA6: "#f92d0050",
    tomatoA7: "#e7280067",
    tomatoA8: "#db250084",
    tomatoA9: "#df2600d1",
    tomatoA10: "#d72400da",
    tomatoA11: "#cd2200ea",
    tomatoA12: "#460900e0",
};
const tomatoP3 = {
    tomato1: "color(display-p3 0.9979 0.9886 0.9884)",
    tomato2: "color(display-p3 0.9938 0.9737 0.9686)",
    tomato3: "color(display-p3 0.9847 0.9243 0.9092)",
    tomato4: "color(display-p3 0.9957 0.8676 0.8354)",
    tomato5: "color(display-p3 0.98 0.8122 0.7699)",
    tomato6: "color(display-p3 0.9531 0.7496 0.6983)",
    tomato7: "color(display-p3 0.9169 0.6727 0.6113)",
    tomato8: "color(display-p3 0.8752 0.5749 0.5015)",
    tomato9: "color(display-p3 0.8309 0.3454 0.2311)",
    tomato10: "color(display-p3 0.802 0.3128 0.2003)",
    tomato11: "color(display-p3 0.7545 0.2585 0.1518)",
    tomato12: "color(display-p3 0.3347 0.1648 0.1322)",
};
const tomatoP3A = {
    tomatoA1: "color(display-p3 0.6745 0.0235 0.0235 / 0.012)",
    tomatoA2: "color(display-p3 0.7569 0.1451 0.0196 / 0.032)",
    tomatoA3: "color(display-p3 0.8314 0.1843 0.0118 / 0.091)",
    tomatoA4: "color(display-p3 0.9765 0.1922 0.0039 / 0.165)",
    tomatoA5: "color(display-p3 0.9176 0.1922 0.0039 / 0.232)",
    tomatoA6: "color(display-p3 0.8471 0.1725 0.0039 / 0.302)",
    tomatoA7: "color(display-p3 0.7882 0.1647 0.0039 / 0.389)",
    tomatoA8: "color(display-p3 0.749 0.1529 0.0039 / 0.499)",
    tomatoA9: "color(display-p3 0.7804 0.149 0 / 0.769)",
    tomatoA10: "color(display-p3 0.7569 0.1412 0 / 0.8)",
    tomatoA11: "color(display-p3 0.7545 0.2585 0.1518)",
    tomatoA12: "color(display-p3 0.3347 0.1648 0.1322)",
};
const red = {
    red1: "#fffcfc",
    red2: "#fff7f7",
    red3: "#feebec",
    red4: "#ffdbdc",
    red5: "#ffcdce",
    red6: "#fdbdbe",
    red7: "#f4a9aa",
    red8: "#eb8e90",
    red9: "#e5484d",
    red10: "#dc3e42",
    red11: "#ce2c31",
    red12: "#641723",
};
const redA = {
    redA1: "#ff000003",
    redA2: "#ff000008",
    redA3: "#f3000d14",
    redA4: "#ff000824",
    redA5: "#ff000632",
    redA6: "#f8000442",
    redA7: "#df000356",
    redA8: "#d2000571",
    redA9: "#db0007b7",
    redA10: "#d10005c1",
    redA11: "#c40006d3",
    redA12: "#55000de8",
};
const redP3 = {
    red1: "color(display-p3 0.9979 0.9886 0.9884)",
    red2: "color(display-p3 0.9948 0.971 0.971)",
    red3: "color(display-p3 0.9845 0.9254 0.9253)",
    red4: "color(display-p3 0.9993 0.866 0.8657)",
    red5: "color(display-p3 0.9837 0.8118 0.8111)",
    red6: "color(display-p3 0.9548 0.7508 0.7493)",
    red7: "color(display-p3 0.915 0.6745 0.672)",
    red8: "color(display-p3 0.8715 0.5751 0.5723)",
    red9: "color(display-p3 0.83 0.3294 0.3239)",
    red10: "color(display-p3 0.7976 0.2935 0.2847)",
    red11: "color(display-p3 0.7438 0.2337 0.2223)",
    red12: "color(display-p3 0.3597 0.115 0.1433)",
};
const redP3A = {
    redA1: "color(display-p3 0.6745 0.0235 0.0235 / 0.012)",
    redA2: "color(display-p3 0.8627 0.0235 0.0235 / 0.028)",
    redA3: "color(display-p3 0.7922 0.0078 0.0078 / 0.075)",
    redA4: "color(display-p3 1 0.0078 0.0078 / 0.134)",
    redA5: "color(display-p3 0.9176 0.0078 0.0078 / 0.189)",
    redA6: "color(display-p3 0.8157 0.0039 0.0039 / 0.251)",
    redA7: "color(display-p3 0.7412 0.0157 0.0039 / 0.33)",
    redA8: "color(display-p3 0.698 0.0118 0.0039 / 0.428)",
    redA9: "color(display-p3 0.749 0.0078 0 / 0.675)",
    redA10: "color(display-p3 0.7137 0.0118 0 / 0.714)",
    redA11: "color(display-p3 0.7438 0.2337 0.2223)",
    redA12: "color(display-p3 0.3597 0.115 0.1433)",
};
const ruby = {
    ruby1: "#fffcfd",
    ruby2: "#fff7f8",
    ruby3: "#feeaed",
    ruby4: "#ffdce1",
    ruby5: "#ffced6",
    ruby6: "#f8bfc8",
    ruby7: "#efacb8",
    ruby8: "#e592a3",
    ruby9: "#e54666",
    ruby10: "#dc3b5d",
    ruby11: "#ca244d",
    ruby12: "#64172b",
};
const rubyA = {
    rubyA1: "#ff005503",
    rubyA2: "#ff002008",
    rubyA3: "#f3002515",
    rubyA4: "#ff002523",
    rubyA5: "#ff002a31",
    rubyA6: "#e4002440",
    rubyA7: "#ce002553",
    rubyA8: "#c300286d",
    rubyA9: "#db002cb9",
    rubyA10: "#d2002cc4",
    rubyA11: "#c10030db",
    rubyA12: "#550016e8",
};
const rubyP3 = {
    ruby1: "color(display-p3 0.9979 0.9886 0.992)",
    ruby2: "color(display-p3 0.9948 0.971 0.9737)",
    ruby3: "color(display-p3 0.9831 0.9202 0.9285)",
    ruby4: "color(display-p3 0.9874 0.8692 0.8851)",
    ruby5: "color(display-p3 0.9681 0.8168 0.8388)",
    ruby6: "color(display-p3 0.9374 0.7581 0.7861)",
    ruby7: "color(display-p3 0.8967 0.6848 0.7212)",
    ruby8: "color(display-p3 0.8507 0.588 0.6393)",
    ruby9: "color(display-p3 0.8296 0.3231 0.4084)",
    ruby10: "color(display-p3 0.795 0.2861 0.3748)",
    ruby11: "color(display-p3 0.728 0.2111 0.3113)",
    ruby12: "color(display-p3 0.3597 0.115 0.1711)",
};
const rubyP3A = {
    rubyA1: "color(display-p3 0.6745 0.0235 0.349 / 0.012)",
    rubyA2: "color(display-p3 0.8627 0.0235 0.0235 / 0.028)",
    rubyA3: "color(display-p3 0.8039 0.0078 0.1098 / 0.079)",
    rubyA4: "color(display-p3 0.9098 0.0078 0.1255 / 0.13)",
    rubyA5: "color(display-p3 0.8314 0.0039 0.1333 / 0.185)",
    rubyA6: "color(display-p3 0.7451 0.0039 0.1176 / 0.244)",
    rubyA7: "color(display-p3 0.6784 0.0039 0.1137 / 0.314)",
    rubyA8: "color(display-p3 0.6392 0.0039 0.1255 / 0.412)",
    rubyA9: "color(display-p3 0.7529 0 0.1294 / 0.679)",
    rubyA10: "color(display-p3 0.7137 0 0.1255 / 0.714)",
    rubyA11: "color(display-p3 0.728 0.2111 0.3113)",
    rubyA12: "color(display-p3 0.3597 0.115 0.1711)",
};
const crimson = {
    crimson1: "#fffcfd",
    crimson2: "#fef7f9",
    crimson3: "#feeaf0",
    crimson4: "#ffdbe8",
    crimson5: "#fdcdde",
    crimson6: "#f5bed2",
    crimson7: "#eaabc3",
    crimson8: "#e093b2",
    crimson9: "#e93d82",
    crimson10: "#df3478",
    crimson11: "#cb1d63",
    crimson12: "#621639",
};
const crimsonA = {
    crimsonA1: "#ff005503",
    crimsonA2: "#e0004008",
    crimsonA3: "#f3004915",
    crimsonA4: "#ff005d24",
    crimsonA5: "#f5005732",
    crimsonA6: "#d8004f41",
    crimsonA7: "#c0004954",
    crimsonA8: "#b6004a6c",
    crimsonA9: "#e2005bc2",
    crimsonA10: "#d70056cb",
    crimsonA11: "#c4004fe2",
    crimsonA12: "#530026e9",
};
const crimsonP3 = {
    crimson1: "color(display-p3 0.9979 0.9886 0.992)",
    crimson2: "color(display-p3 0.9911 0.9689 0.9761)",
    crimson3: "color(display-p3 0.9835 0.9221 0.9419)",
    crimson4: "color(display-p3 0.9859 0.8651 0.9076)",
    crimson5: "color(display-p3 0.9629 0.8124 0.8672)",
    crimson6: "color(display-p3 0.9267 0.7537 0.8192)",
    crimson7: "color(display-p3 0.8817 0.6821 0.7611)",
    crimson8: "color(display-p3 0.8345 0.5916 0.6939)",
    crimson9: "color(display-p3 0.8425 0.2978 0.507)",
    crimson10: "color(display-p3 0.8069 0.2656 0.4682)",
    crimson11: "color(display-p3 0.731 0.1954 0.3881)",
    crimson12: "color(display-p3 0.3523 0.111 0.2207)",
};
const crimsonP3A = {
    crimsonA1: "color(display-p3 0.6745 0.0235 0.349 / 0.012)",
    crimsonA2: "color(display-p3 0.7569 0.0196 0.2667 / 0.032)",
    crimsonA3: "color(display-p3 0.8039 0.0078 0.2588 / 0.079)",
    crimsonA4: "color(display-p3 0.8863 0.0078 0.298 / 0.134)",
    crimsonA5: "color(display-p3 0.8157 0.0078 0.298 / 0.189)",
    crimsonA6: "color(display-p3 0.702 0.0039 0.2745 / 0.248)",
    crimsonA7: "color(display-p3 0.6314 0.0039 0.251 / 0.318)",
    crimsonA8: "color(display-p3 0.6 0.0039 0.251 / 0.408)",
    crimsonA9: "color(display-p3 0.7765 0 0.298 / 0.702)",
    crimsonA10: "color(display-p3 0.7373 0 0.2745 / 0.734)",
    crimsonA11: "color(display-p3 0.731 0.1954 0.3881)",
    crimsonA12: "color(display-p3 0.3523 0.111 0.2207)",
};
const pink = {
    pink1: "#fffcfe",
    pink2: "#fef7fb",
    pink3: "#feebf6",
    pink4: "#fbddef",
    pink5: "#f6d0e7",
    pink6: "#efc0dd",
    pink7: "#e7acd1",
    pink8: "#dd93c2",
    pink9: "#d6409f",
    pink10: "#cf3897",
    pink11: "#c2298a",
    pink12: "#651249",
};
const pinkA = {
    pinkA1: "#ff00aa03",
    pinkA2: "#e0008008",
    pinkA3: "#f3008d14",
    pinkA4: "#e1008722",
    pinkA5: "#cf007d2f",
    pinkA6: "#bf00763f",
    pinkA7: "#b6007253",
    pinkA8: "#af006f6c",
    pinkA9: "#c8007fbf",
    pinkA10: "#c2007ac7",
    pinkA11: "#b60074d6",
    pinkA12: "#59003bed",
};
const pinkP3 = {
    pink1: "color(display-p3 0.9979 0.9886 0.9956)",
    pink2: "color(display-p3 0.9924 0.9695 0.9848)",
    pink3: "color(display-p3 0.9839 0.9239 0.9639)",
    pink4: "color(display-p3 0.9643 0.8718 0.9342)",
    pink5: "color(display-p3 0.94 0.8197 0.9011)",
    pink6: "color(display-p3 0.9082 0.7588 0.8601)",
    pink7: "color(display-p3 0.8695 0.6836 0.8102)",
    pink8: "color(display-p3 0.8247 0.5901 0.7512)",
    pink9: "color(display-p3 0.7749 0.2973 0.6104)",
    pink10: "color(display-p3 0.748 0.2703 0.5813)",
    pink11: "color(display-p3 0.6981 0.2189 0.528)",
    pink12: "color(display-p3 0.3625 0.1009 0.2791)",
};
const pinkP3A = {
    pinkA1: "color(display-p3 0.6745 0.0235 0.6745 / 0.012)",
    pinkA2: "color(display-p3 0.7569 0.0196 0.5098 / 0.032)",
    pinkA3: "color(display-p3 0.7922 0.0078 0.5294 / 0.075)",
    pinkA4: "color(display-p3 0.7294 0.0078 0.4902 / 0.13)",
    pinkA5: "color(display-p3 0.6784 0.0039 0.4588 / 0.181)",
    pinkA6: "color(display-p3 0.6314 0.0039 0.4235 / 0.244)",
    pinkA7: "color(display-p3 0.5961 0.0039 0.4118 / 0.318)",
    pinkA8: "color(display-p3 0.5725 0.0039 0.4039 / 0.412)",
    pinkA9: "color(display-p3 0.6824 0 0.4471 / 0.702)",
    pinkA10: "color(display-p3 0.6549 0 0.4235 / 0.73)",
    pinkA11: "color(display-p3 0.6981 0.2189 0.528)",
    pinkA12: "color(display-p3 0.3625 0.1009 0.2791)",
};
const plum = {
    plum1: "#fefcff",
    plum2: "#fff6ff",
    plum3: "#fdebfe",
    plum4: "#fadcfc",
    plum5: "#f4d0f7",
    plum6: "#ebc1ef",
    plum7: "#dfaee5",
    plum8: "#cf91d8",
    plum9: "#ab4aba",
    plum10: "#a144af",
    plum11: "#953ea3",
    plum12: "#53195d",
};
const plumA = {
    plumA1: "#aa00ff03",
    plumA2: "#ff00ff09",
    plumA3: "#e600f314",
    plumA4: "#db00ea23",
    plumA5: "#c400d42f",
    plumA6: "#ad00be3e",
    plumA7: "#9b00ae51",
    plumA8: "#9000a56e",
    plumA9: "#89009eb5",
    plumA10: "#7f0092bb",
    plumA11: "#730086c1",
    plumA12: "#40004be6",
};
const plumP3 = {
    plum1: "color(display-p3 0.9947 0.9885 0.9991)",
    plum2: "color(display-p3 0.9931 0.9671 0.9976)",
    plum3: "color(display-p3 0.9798 0.9253 0.99)",
    plum4: "color(display-p3 0.9606 0.8676 0.9793)",
    plum5: "color(display-p3 0.9338 0.8193 0.9584)",
    plum6: "color(display-p3 0.8966 0.7641 0.9271)",
    plum7: "color(display-p3 0.8451 0.6896 0.8843)",
    plum8: "color(display-p3 0.7755 0.5795 0.8318)",
    plum9: "color(display-p3 0.6238 0.3128 0.7077)",
    plum10: "color(display-p3 0.5874 0.2901 0.6673)",
    plum11: "color(display-p3 0.5434 0.2628 0.6186)",
    plum12: "color(display-p3 0.2993 0.1136 0.3523)",
};
const plumP3A = {
    plumA1: "color(display-p3 0.6745 0.0235 1 / 0.012)",
    plumA2: "color(display-p3 0.7569 0.0196 0.8784 / 0.032)",
    plumA3: "color(display-p3 0.7412 0.0078 0.8431 / 0.075)",
    plumA4: "color(display-p3 0.7098 0.0078 0.8549 / 0.134)",
    plumA5: "color(display-p3 0.6353 0.0039 0.7647 / 0.181)",
    plumA6: "color(display-p3 0.5686 0.0039 0.6863 / 0.236)",
    plumA7: "color(display-p3 0.5098 0.0039 0.6235 / 0.31)",
    plumA8: "color(display-p3 0.4706 0.0039 0.6 / 0.42)",
    plumA9: "color(display-p3 0.451 0 0.5725 / 0.687)",
    plumA10: "color(display-p3 0.4196 0 0.5294 / 0.71)",
    plumA11: "color(display-p3 0.5434 0.2628 0.6186)",
    plumA12: "color(display-p3 0.2993 0.1136 0.3523)",
};
const purple = {
    purple1: "#fefcfe",
    purple2: "#fcf7ff",
    purple3: "#f8edfe",
    purple4: "#f4dfff",
    purple5: "#edd2fd",
    purple6: "#e3c4f7",
    purple7: "#d3b0ee",
    purple8: "#be93e4",
    purple9: "#8e4ec6",
    purple10: "#8347b9",
    purple11: "#8145b5",
    purple12: "#402060",
};
const purpleA = {
    purpleA1: "#aa00aa03",
    purpleA2: "#a000ff08",
    purpleA3: "#9c00f112",
    purpleA4: "#a800ff20",
    purpleA5: "#9900f42d",
    purpleA6: "#8601dd3b",
    purpleA7: "#7100c94f",
    purpleA8: "#6600c06c",
    purpleA9: "#5c00adb1",
    purpleA10: "#53009eb8",
    purpleA11: "#52009aba",
    purpleA12: "#250049df",
};
const purpleP3 = {
    purple1: "color(display-p3 0.9947 0.9885 0.9955)",
    purple2: "color(display-p3 0.9856 0.9687 0.9975)",
    purple3: "color(display-p3 0.9665 0.9324 0.9921)",
    purple4: "color(display-p3 0.9435 0.8771 0.996)",
    purple5: "color(display-p3 0.9121 0.8286 0.9819)",
    purple6: "color(display-p3 0.8687 0.7715 0.9556)",
    purple7: "color(display-p3 0.8066 0.6941 0.9168)",
    purple8: "color(display-p3 0.7192 0.5827 0.8736)",
    purple9: "color(display-p3 0.5231 0.3183 0.7512)",
    purple10: "color(display-p3 0.4832 0.2887 0.6997)",
    purple11: "color(display-p3 0.4733 0.2813 0.687)",
    purple12: "color(display-p3 0.2341 0.1317 0.363)",
};
const purpleP3A = {
    purpleA1: "color(display-p3 0.6745 0.0235 0.6745 / 0.012)",
    purpleA2: "color(display-p3 0.5098 0.0196 0.8784 / 0.032)",
    purpleA3: "color(display-p3 0.4745 0.0078 0.8863 / 0.067)",
    purpleA4: "color(display-p3 0.5529 0.0039 0.9686 / 0.122)",
    purpleA5: "color(display-p3 0.502 0.0039 0.8902 / 0.173)",
    purpleA6: "color(display-p3 0.4353 0.0039 0.8118 / 0.228)",
    purpleA7: "color(display-p3 0.3725 0.0039 0.7333 / 0.306)",
    purpleA8: "color(display-p3 0.3216 0.0039 0.702 / 0.416)",
    purpleA9: "color(display-p3 0.298 0 0.6392 / 0.683)",
    purpleA10: "color(display-p3 0.2706 0 0.5765 / 0.71)",
    purpleA11: "color(display-p3 0.4733 0.2813 0.687)",
    purpleA12: "color(display-p3 0.2341 0.1317 0.363)",
};
const violet = {
    violet1: "#fdfcfe",
    violet2: "#faf8ff",
    violet3: "#f4f0fe",
    violet4: "#ebe4ff",
    violet5: "#e1d9ff",
    violet6: "#d4cafe",
    violet7: "#c2b5f5",
    violet8: "#aa99ec",
    violet9: "#6e56cf",
    violet10: "#654dc4",
    violet11: "#6550b9",
    violet12: "#2f265f",
};
const violetA = {
    violetA1: "#5500aa03",
    violetA2: "#4900ff07",
    violetA3: "#4400ee0f",
    violetA4: "#4300ff1b",
    violetA5: "#3600ff26",
    violetA6: "#3100fb35",
    violetA7: "#2d01dd4a",
    violetA8: "#2b00d066",
    violetA9: "#2400b7a9",
    violetA10: "#2300abb2",
    violetA11: "#1f0099af",
    violetA12: "#0b0043d9",
};
const violetP3 = {
    violet1: "color(display-p3 0.9915 0.9884 0.9954)",
    violet2: "color(display-p3 0.9783 0.9742 0.9978)",
    violet3: "color(display-p3 0.9534 0.9434 0.9925)",
    violet4: "color(display-p3 0.916 0.8965 1)",
    violet5: "color(display-p3 0.8765 0.8505 1)",
    violet6: "color(display-p3 0.8251 0.7928 0.9809)",
    violet7: "color(display-p3 0.7522 0.7121 0.9425)",
    violet8: "color(display-p3 0.654 0.6025 0.9016)",
    violet9: "color(display-p3 0.4166 0.3409 0.7843)",
    violet10: "color(display-p3 0.381 0.3061 0.741)",
    violet11: "color(display-p3 0.3831 0.3168 0.7021)",
    violet12: "color(display-p3 0.1786 0.1503 0.3593)",
};
const violetP3A = {
    violetA1: "color(display-p3 0.349 0.0235 0.6745 / 0.012)",
    violetA2: "color(display-p3 0.1608 0.0235 0.8627 / 0.028)",
    violetA3: "color(display-p3 0.1451 0.0039 0.8588 / 0.055)",
    violetA4: "color(display-p3 0.1961 0.0039 1 / 0.102)",
    violetA5: "color(display-p3 0.1922 0.0078 1 / 0.15)",
    violetA6: "color(display-p3 0.1529 0.0039 0.9059 / 0.208)",
    violetA7: "color(display-p3 0.1412 0.0039 0.7961 / 0.287)",
    violetA8: "color(display-p3 0.1333 0.0039 0.7529 / 0.397)",
    violetA9: "color(display-p3 0.1137 0 0.6745 / 0.659)",
    violetA10: "color(display-p3 0.1098 0 0.6275 / 0.695)",
    violetA11: "color(display-p3 0.3831 0.3168 0.7021)",
    violetA12: "color(display-p3 0.1786 0.1503 0.3593)",
};
const iris = {
    iris1: "#fdfdff",
    iris2: "#f8f8ff",
    iris3: "#f0f1fe",
    iris4: "#e6e7ff",
    iris5: "#dadcff",
    iris6: "#cbcdff",
    iris7: "#b8baf8",
    iris8: "#9b9ef0",
    iris9: "#5b5bd6",
    iris10: "#5151cd",
    iris11: "#5753c6",
    iris12: "#272962",
};
const irisA = {
    irisA1: "#0000ff02",
    irisA2: "#0000ff07",
    irisA3: "#0011ee0f",
    irisA4: "#000bff19",
    irisA5: "#000eff25",
    irisA6: "#000aff34",
    irisA7: "#0008e647",
    irisA8: "#0008d964",
    irisA9: "#0000c0a4",
    irisA10: "#0000b6ae",
    irisA11: "#0600abac",
    irisA12: "#000246d8",
};
const irisP3 = {
    iris1: "color(display-p3 0.9922 0.9922 0.9993)",
    iris2: "color(display-p3 0.9722 0.9729 0.9976)",
    iris3: "color(display-p3 0.9433 0.9447 0.9924)",
    iris4: "color(display-p3 0.9022 0.9057 1)",
    iris5: "color(display-p3 0.8568 0.8614 1)",
    iris6: "color(display-p3 0.7992 0.8046 0.9871)",
    iris7: "color(display-p3 0.7212 0.7273 0.9551)",
    iris8: "color(display-p3 0.6105 0.6192 0.9184)",
    iris9: "color(display-p3 0.3569 0.3569 0.8104)",
    iris10: "color(display-p3 0.318 0.318 0.7737)",
    iris11: "color(display-p3 0.3374 0.3255 0.7483)",
    iris12: "color(display-p3 0.1544 0.1605 0.3706)",
};
const irisP3A = {
    irisA1: "color(display-p3 0.0196 0.0196 1 / 0.008)",
    irisA2: "color(display-p3 0.0235 0.0235 0.8627 / 0.028)",
    irisA3: "color(display-p3 0.0039 0.0039 0.8588 / 0.055)",
    irisA4: "color(display-p3 0.0118 0.051 1 / 0.099)",
    irisA5: "color(display-p3 0.0078 0.0627 1 / 0.146)",
    irisA6: "color(display-p3 0 0.0196 0.9412 / 0.2)",
    irisA7: "color(display-p3 0.0039 0.0196 0.8471 / 0.279)",
    irisA8: "color(display-p3 0.0039 0.0235 0.7882 / 0.389)",
    irisA9: "color(display-p3 0 0 0.7059 / 0.644)",
    irisA10: "color(display-p3 0 0 0.6667 / 0.683)",
    irisA11: "color(display-p3 0.3374 0.3255 0.7483)",
    irisA12: "color(display-p3 0.1544 0.1605 0.3706)",
};
const indigo = {
    indigo1: "#fdfdfe",
    indigo2: "#f7f9ff",
    indigo3: "#edf2fe",
    indigo4: "#e1e9ff",
    indigo5: "#d2deff",
    indigo6: "#c1d0ff",
    indigo7: "#abbdf9",
    indigo8: "#8da4ef",
    indigo9: "#3e63dd",
    indigo10: "#3358d4",
    indigo11: "#3a5bc7",
    indigo12: "#1f2d5c",
};
const indigoA = {
    indigoA1: "#00008002",
    indigoA2: "#0040ff08",
    indigoA3: "#0047f112",
    indigoA4: "#0044ff1e",
    indigoA5: "#0044ff2d",
    indigoA6: "#003eff3e",
    indigoA7: "#0037ed54",
    indigoA8: "#0034dc72",
    indigoA9: "#0031d2c1",
    indigoA10: "#002ec9cc",
    indigoA11: "#002bb7c5",
    indigoA12: "#001046e0",
};
const indigoP3 = {
    indigo1: "color(display-p3 0.9922 0.9922 0.9957)",
    indigo2: "color(display-p3 0.9712 0.9768 0.9979)",
    indigo3: "color(display-p3 0.9331 0.948 0.9919)",
    indigo4: "color(display-p3 0.885 0.9139 1)",
    indigo5: "color(display-p3 0.8313 0.8703 1)",
    indigo6: "color(display-p3 0.7673 0.8143 0.9952)",
    indigo7: "color(display-p3 0.6849 0.7399 0.9574)",
    indigo8: "color(display-p3 0.5686 0.6389 0.9163)",
    indigo9: "color(display-p3 0.2757 0.3844 0.8369)",
    indigo10: "color(display-p3 0.2339 0.3431 0.8009)",
    indigo11: "color(display-p3 0.2565 0.3537 0.7554)",
    indigo12: "color(display-p3 0.133 0.1749 0.3485)",
};
const indigoP3A = {
    indigoA1: "color(display-p3 0.0196 0.0196 0.5098 / 0.008)",
    indigoA2: "color(display-p3 0.0235 0.1608 0.8627 / 0.028)",
    indigoA3: "color(display-p3 0.0078 0.2392 0.8863 / 0.067)",
    indigoA4: "color(display-p3 0.0039 0.2471 1 / 0.114)",
    indigoA5: "color(display-p3 0.0039 0.2353 1 / 0.169)",
    indigoA6: "color(display-p3 0.0039 0.2078 0.9843 / 0.232)",
    indigoA7: "color(display-p3 0.0039 0.1765 0.8627 / 0.314)",
    indigoA8: "color(display-p3 0.0039 0.1647 0.8118 / 0.432)",
    indigoA9: "color(display-p3 0 0.1529 0.7725 / 0.726)",
    indigoA10: "color(display-p3 0 0.1373 0.7373 / 0.765)",
    indigoA11: "color(display-p3 0.2565 0.3537 0.7554)",
    indigoA12: "color(display-p3 0.133 0.1749 0.3485)",
};
const blue = {
    blue1: "#fbfdff",
    blue2: "#f4faff",
    blue3: "#e6f4fe",
    blue4: "#d5efff",
    blue5: "#c2e5ff",
    blue6: "#acd8fc",
    blue7: "#8ec8f6",
    blue8: "#5eb1ef",
    blue9: "#0090ff",
    blue10: "#0588f0",
    blue11: "#0d74ce",
    blue12: "#113264",
};
const blueA = {
    blueA1: "#0080ff04",
    blueA2: "#008cff0b",
    blueA3: "#008ff519",
    blueA4: "#009eff2a",
    blueA5: "#0093ff3d",
    blueA6: "#0088f653",
    blueA7: "#0083eb71",
    blueA8: "#0084e6a1",
    blueA9: "#0090ff",
    blueA10: "#0086f0fa",
    blueA11: "#006dcbf2",
    blueA12: "#002359ee",
};
const blueP3 = {
    blue1: "color(display-p3 0.9857 0.9919 0.9992)",
    blue2: "color(display-p3 0.9604 0.9794 0.9979)",
    blue3: "color(display-p3 0.9115 0.9556 0.9906)",
    blue4: "color(display-p3 0.8534 0.9324 1)",
    blue5: "color(display-p3 0.788 0.8939 0.9978)",
    blue6: "color(display-p3 0.7089 0.843 0.9762)",
    blue7: "color(display-p3 0.6059 0.7766 0.947)",
    blue8: "color(display-p3 0.4509 0.6878 0.9166)",
    blue9: "color(display-p3 0.2466 0.5561 0.9689)",
    blue10: "color(display-p3 0.2338 0.5233 0.9124)",
    blue11: "color(display-p3 0.15 0.44 0.84)",
    blue12: "color(display-p3 0.1016 0.1932 0.3787)",
};
const blueP3A = {
    blueA1: "color(display-p3 0.0235 0.5137 1 / 0.016)",
    blueA2: "color(display-p3 0.0235 0.5137 0.9059 / 0.04)",
    blueA3: "color(display-p3 0.0118 0.5294 0.9176 / 0.091)",
    blueA4: "color(display-p3 0.0078 0.5451 1 / 0.146)",
    blueA5: "color(display-p3 0.0039 0.502 0.9843 / 0.212)",
    blueA6: "color(display-p3 0.0039 0.4627 0.9216 / 0.291)",
    blueA7: "color(display-p3 0.0039 0.4314 0.8627 / 0.393)",
    blueA8: "color(display-p3 0 0.4275 0.851 / 0.55)",
    blueA9: "color(display-p3 0 0.4118 0.9608 / 0.753)",
    blueA10: "color(display-p3 0 0.3765 0.8863 / 0.765)",
    blueA11: "color(display-p3 0.15 0.44 0.84)",
    blueA12: "color(display-p3 0.1016 0.1932 0.3787)",
};
const cyan = {
    cyan1: "#fafdfe",
    cyan2: "#f0fbfc",
    cyan3: "#e1f8fa",
    cyan4: "#ccf2f6",
    cyan5: "#b7e9ef",
    cyan6: "#9cdde6",
    cyan7: "#7acddb",
    cyan8: "#3db9cf",
    cyan9: "#00a2c7",
    cyan10: "#0797b9",
    cyan11: "#107d98",
    cyan12: "#0d3c48",
};
const cyanA = {
    cyanA1: "#0099cc05",
    cyanA2: "#00bbcc0f",
    cyanA3: "#00c4d51e",
    cyanA4: "#00bed233",
    cyanA5: "#00b2c748",
    cyanA6: "#00a8bf63",
    cyanA7: "#00a0ba85",
    cyanA8: "#00a3c0c2",
    cyanA9: "#00a2c7",
    cyanA10: "#0094b7f8",
    cyanA11: "#007491ef",
    cyanA12: "#00323ef2",
};
const cyanP3 = {
    cyan1: "color(display-p3 0.9825 0.9918 0.9955)",
    cyan2: "color(display-p3 0.9501 0.9844 0.9887)",
    cyan3: "color(display-p3 0.8981 0.9696 0.977)",
    cyan4: "color(display-p3 0.8296 0.9437 0.9614)",
    cyan5: "color(display-p3 0.756 0.9073 0.9339)",
    cyan6: "color(display-p3 0.6675 0.8584 0.8967)",
    cyan7: "color(display-p3 0.554 0.7951 0.8504)",
    cyan8: "color(display-p3 0.3878 0.7147 0.7984)",
    cyan9: "color(display-p3 0.2818 0.6273 0.7647)",
    cyan10: "color(display-p3 0.2639 0.5833 0.71)",
    cyan11: "color(display-p3 0.08 0.48 0.63)",
    cyan12: "color(display-p3 0.1081 0.2316 0.277)",
};
const cyanP3A = {
    cyanA1: "color(display-p3 0.0235 0.5137 0.7569 / 0.016)",
    cyanA2: "color(display-p3 0.0039 0.6941 0.7725 / 0.051)",
    cyanA3: "color(display-p3 0.0039 0.6941 0.7725 / 0.102)",
    cyanA4: "color(display-p3 0.0039 0.6784 0.7686 / 0.169)",
    cyanA5: "color(display-p3 0.0039 0.6157 0.7294 / 0.244)",
    cyanA6: "color(display-p3 0.0039 0.5804 0.698 / 0.334)",
    cyanA7: "color(display-p3 0.0039 0.5451 0.6667 / 0.448)",
    cyanA8: "color(display-p3 0 0.5333 0.6745 / 0.612)",
    cyanA9: "color(display-p3 0 0.4824 0.6745 / 0.718)",
    cyanA10: "color(display-p3 0 0.4353 0.6078 / 0.738)",
    cyanA11: "color(display-p3 0.08 0.48 0.63)",
    cyanA12: "color(display-p3 0.1081 0.2316 0.277)",
};
const teal = {
    teal1: "#fafefd",
    teal2: "#f0fcf9",
    teal3: "#def9f2",
    teal4: "#cbf4e9",
    teal5: "#b7ebdf",
    teal6: "#a0dfd1",
    teal7: "#83cec0",
    teal8: "#53b9ab",
    teal9: "#12a594",
    teal10: "#0d9b8a",
    teal11: "#008573",
    teal12: "#0d3d38",
};
const tealA = {
    tealA1: "#00cc9905",
    tealA2: "#00cc990f",
    tealA3: "#00d19b21",
    tealA4: "#00ca9434",
    tealA5: "#00b98e48",
    tealA6: "#00aa845f",
    tealA7: "#009b7e7c",
    tealA8: "#009783ac",
    tealA9: "#009e8ced",
    tealA10: "#009684f2",
    tealA11: "#008573",
    tealA12: "#00332df2",
};
const tealP3 = {
    teal1: "color(display-p3 0.9832 0.9956 0.9922)",
    teal2: "color(display-p3 0.9509 0.9883 0.976)",
    teal3: "color(display-p3 0.8923 0.9743 0.9503)",
    teal4: "color(display-p3 0.8278 0.9522 0.9158)",
    teal5: "color(display-p3 0.7596 0.9163 0.874)",
    teal6: "color(display-p3 0.6806 0.8659 0.821)",
    teal7: "color(display-p3 0.5797 0.7991 0.7538)",
    teal8: "color(display-p3 0.4328 0.7156 0.6713)",
    teal9: "color(display-p3 0.297 0.6374 0.5811)",
    teal10: "color(display-p3 0.2752 0.5993 0.5417)",
    teal11: "color(display-p3 0.08 0.5 0.43)",
    teal12: "color(display-p3 0.1097 0.2355 0.2193)",
};
const tealP3A = {
    tealA1: "color(display-p3 0.0235 0.7569 0.5137 / 0.016)",
    tealA2: "color(display-p3 0.0039 0.7725 0.5412 / 0.051)",
    tealA3: "color(display-p3 0.0039 0.7412 0.5216 / 0.106)",
    tealA4: "color(display-p3 0.0039 0.7294 0.5255 / 0.173)",
    tealA5: "color(display-p3 0.0039 0.6588 0.4784 / 0.24)",
    tealA6: "color(display-p3 0.0039 0.5843 0.4353 / 0.318)",
    tealA7: "color(display-p3 0.0039 0.5255 0.4118 / 0.42)",
    tealA8: "color(display-p3 0 0.498 0.4235 / 0.569)",
    tealA9: "color(display-p3 0 0.4863 0.4039 / 0.702)",
    tealA10: "color(display-p3 0 0.451 0.3686 / 0.726)",
    tealA11: "color(display-p3 0.08 0.5 0.43)",
    tealA12: "color(display-p3 0.1097 0.2355 0.2193)",
};
const jade = {
    jade1: "#fbfefd",
    jade2: "#effdf5",
    jade3: "#e1f9eb",
    jade4: "#d2f3e1",
    jade5: "#c0ead6",
    jade6: "#a9dfc8",
    jade7: "#89cfb5",
    jade8: "#56ba9f",
    jade9: "#29a383",
    jade10: "#26997b",
    jade11: "#208368",
    jade12: "#1d3b31",
};
const jadeA = {
    jadeA1: "#00c08004",
    jadeA2: "#00e06010",
    jadeA3: "#00cc551e",
    jadeA4: "#00bb552d",
    jadeA5: "#00aa5a3f",
    jadeA6: "#00a15c56",
    jadeA7: "#00986076",
    jadeA8: "#00976ea9",
    jadeA9: "#00916bd6",
    jadeA10: "#008764d9",
    jadeA11: "#007152df",
    jadeA12: "#002217e2",
};
const jadeP3 = {
    jade1: "color(display-p3 0.9864 0.9957 0.9923)",
    jade2: "color(display-p3 0.9468 0.9919 0.9623)",
    jade3: "color(display-p3 0.8994 0.9722 0.9266)",
    jade4: "color(display-p3 0.8469 0.9471 0.8879)",
    jade5: "color(display-p3 0.7853 0.914 0.8433)",
    jade6: "color(display-p3 0.707 0.8683 0.7883)",
    jade7: "color(display-p3 0.598 0.8042 0.7169)",
    jade8: "color(display-p3 0.44 0.7197 0.6286)",
    jade9: "color(display-p3 0.3195 0.6301 0.5205)",
    jade10: "color(display-p3 0.299 0.5919 0.4882)",
    jade11: "color(display-p3 0.15 0.5 0.37)",
    jade12: "color(display-p3 0.1422 0.2285 0.1942)",
};
const jadeP3A = {
    jadeA1: "color(display-p3 0.0235 0.6745 0.349 / 0.012)",
    jadeA2: "color(display-p3 0.0039 0.8588 0.2902 / 0.055)",
    jadeA3: "color(display-p3 0.0039 0.7333 0.2706 / 0.102)",
    jadeA4: "color(display-p3 0.0039 0.6706 0.2588 / 0.153)",
    jadeA5: "color(display-p3 0.0039 0.6039 0.2745 / 0.216)",
    jadeA6: "color(display-p3 0.0039 0.549 0.2824 / 0.295)",
    jadeA7: "color(display-p3 0.0039 0.5176 0.302 / 0.404)",
    jadeA8: "color(display-p3 0 0.5059 0.3373 / 0.561)",
    jadeA9: "color(display-p3 0 0.4588 0.298 / 0.683)",
    jadeA10: "color(display-p3 0 0.4196 0.2706 / 0.702)",
    jadeA11: "color(display-p3 0.15 0.5 0.37)",
    jadeA12: "color(display-p3 0.1422 0.2285 0.1942)",
};
const green = {
    green1: "#fbfefc",
    green2: "#f2fcf5",
    green3: "#e4f8ea",
    green4: "#d5f2de",
    green5: "#c3e9d0",
    green6: "#acdebf",
    green7: "#8dcea9",
    green8: "#5bb98b",
    green9: "#30a46c",
    green10: "#2b9a66",
    green11: "#218358",
    green12: "#193b2d",
};
const greenA = {
    greenA1: "#00c04004",
    greenA2: "#00c53b0d",
    greenA3: "#00bd391b",
    greenA4: "#00b1372a",
    greenA5: "#00a2383c",
    greenA6: "#009a3b53",
    greenA7: "#00923f72",
    greenA8: "#00924ba4",
    greenA9: "#008f4acf",
    greenA10: "#008647d4",
    greenA11: "#00713fde",
    greenA12: "#002616e6",
};
const greenP3 = {
    green1: "color(display-p3 0.9864 0.9957 0.9887)",
    green2: "color(display-p3 0.9574 0.9885 0.9627)",
    green3: "color(display-p3 0.9088 0.9691 0.9204)",
    green4: "color(display-p3 0.8554 0.9438 0.8745)",
    green5: "color(display-p3 0.7935 0.9104 0.8221)",
    green6: "color(display-p3 0.7159 0.865 0.7577)",
    green7: "color(display-p3 0.6092 0.802 0.6731)",
    green8: "color(display-p3 0.4509 0.7155 0.559)",
    green9: "color(display-p3 0.3322 0.6342 0.4416)",
    green10: "color(display-p3 0.3078 0.5948 0.4168)",
    green11: "color(display-p3 0.19 0.5 0.32)",
    green12: "color(display-p3 0.1322 0.2283 0.18)",
};
const greenP3A = {
    greenA1: "color(display-p3 0.0235 0.6745 0.0235 / 0.012)",
    greenA2: "color(display-p3 0.0196 0.7333 0.1098 / 0.044)",
    greenA3: "color(display-p3 0.0118 0.6588 0.1412 / 0.091)",
    greenA4: "color(display-p3 0.0078 0.6275 0.1412 / 0.146)",
    greenA5: "color(display-p3 0.0039 0.5686 0.1529 / 0.208)",
    greenA6: "color(display-p3 0.0039 0.5294 0.1412 / 0.283)",
    greenA7: "color(display-p3 0.0039 0.502 0.1725 / 0.393)",
    greenA8: "color(display-p3 0 0.4784 0.2 / 0.55)",
    greenA9: "color(display-p3 0 0.4549 0.1647 / 0.667)",
    greenA10: "color(display-p3 0 0.4196 0.1608 / 0.695)",
    greenA11: "color(display-p3 0.19 0.5 0.32)",
    greenA12: "color(display-p3 0.1322 0.2283 0.18)",
};
const grass = {
    grass1: "#fbfefb",
    grass2: "#f3fcf3",
    grass3: "#e6f7e7",
    grass4: "#d8f1d9",
    grass5: "#c6e9c9",
    grass6: "#b0deb5",
    grass7: "#92cf9a",
    grass8: "#65ba74",
    grass9: "#46a758",
    grass10: "#3e9b4f",
    grass11: "#2a7e3b",
    grass12: "#203c25",
};
const grassA = {
    grassA1: "#00c00004",
    grassA2: "#00c0000c",
    grassA3: "#00ae0b19",
    grassA4: "#00a40727",
    grassA5: "#009d0e39",
    grassA6: "#0095114f",
    grassA7: "#008f136d",
    grassA8: "#008d199a",
    grassA9: "#008619b9",
    grassA10: "#007b17c1",
    grassA11: "#006514d5",
    grassA12: "#002006df",
};
const grassP3 = {
    grass1: "color(display-p3 0.9864 0.9957 0.9852)",
    grass2: "color(display-p3 0.9585 0.9868 0.9563)",
    grass3: "color(display-p3 0.9151 0.9684 0.9113)",
    grass4: "color(display-p3 0.8651 0.9437 0.8601)",
    grass5: "color(display-p3 0.8046 0.9105 0.7989)",
    grass6: "color(display-p3 0.7278 0.8657 0.7222)",
    grass7: "color(display-p3 0.6243 0.8045 0.6209)",
    grass8: "color(display-p3 0.4773 0.7204 0.4822)",
    grass9: "color(display-p3 0.3795 0.6466 0.3779)",
    grass10: "color(display-p3 0.3441 0.5982 0.3419)",
    grass11: "color(display-p3 0.2631 0.4882 0.2607)",
    grass12: "color(display-p3 0.1513 0.2325 0.1533)",
};
const grassP3A = {
    grassA1: "color(display-p3 0.2667 0.7569 0.0235 / 0.016)",
    grassA2: "color(display-p3 0.0196 0.7333 0.0196 / 0.044)",
    grassA3: "color(display-p3 0.0549 0.6588 0.0118 / 0.091)",
    grassA4: "color(display-p3 0.0627 0.6157 0.0078 / 0.142)",
    grassA5: "color(display-p3 0.0196 0.549 0 / 0.2)",
    grassA6: "color(display-p3 0.0314 0.5255 0.0039 / 0.279)",
    grassA7: "color(display-p3 0.0118 0.4863 0.0039 / 0.381)",
    grassA8: "color(display-p3 0 0.4667 0.0078 / 0.522)",
    grassA9: "color(display-p3 0.0078 0.4353 0 / 0.624)",
    grassA10: "color(display-p3 0.0078 0.3922 0 / 0.659)",
    grassA11: "color(display-p3 0.2631 0.4882 0.2607)",
    grassA12: "color(display-p3 0.1513 0.2325 0.1533)",
};
const brown = {
    brown1: "#fefdfc",
    brown2: "#fcf9f6",
    brown3: "#f6eee7",
    brown4: "#f0e4d9",
    brown5: "#ebdaca",
    brown6: "#e4cdb7",
    brown7: "#dcbc9f",
    brown8: "#cea37e",
    brown9: "#ad7f58",
    brown10: "#a07553",
    brown11: "#815e46",
    brown12: "#3e332e",
};
const brownA = {
    brownA1: "#aa550003",
    brownA2: "#aa550009",
    brownA3: "#a04b0018",
    brownA4: "#9b4a0026",
    brownA5: "#9f4d0035",
    brownA6: "#a04e0048",
    brownA7: "#a34e0060",
    brownA8: "#9f4a0081",
    brownA9: "#823c00a7",
    brownA10: "#723300ac",
    brownA11: "#522100b9",
    brownA12: "#140600d1",
};
const brownP3 = {
    brown1: "color(display-p3 0.9954 0.9923 0.9887)",
    brown2: "color(display-p3 0.9869 0.9764 0.9642)",
    brown3: "color(display-p3 0.9593 0.9357 0.9087)",
    brown4: "color(display-p3 0.9341 0.8967 0.8547)",
    brown5: "color(display-p3 0.9091 0.8561 0.7977)",
    brown6: "color(display-p3 0.8803 0.8077 0.7296)",
    brown7: "color(display-p3 0.8407 0.7418 0.6387)",
    brown8: "color(display-p3 0.7817 0.647 0.5138)",
    brown9: "color(display-p3 0.651 0.5054 0.3677)",
    brown10: "color(display-p3 0.6009 0.465 0.344)",
    brown11: "color(display-p3 0.485 0.3742 0.2884)",
    brown12: "color(display-p3 0.2361 0.2016 0.1832)",
};
const brownP3A = {
    brownA1: "color(display-p3 0.6745 0.349 0.0235 / 0.012)",
    brownA2: "color(display-p3 0.6745 0.349 0.0235 / 0.036)",
    brownA3: "color(display-p3 0.5725 0.3137 0.0118 / 0.091)",
    brownA4: "color(display-p3 0.5451 0.302 0.0078 / 0.146)",
    brownA5: "color(display-p3 0.5608 0.2902 0.0039 / 0.204)",
    brownA6: "color(display-p3 0.5529 0.2941 0.0039 / 0.271)",
    brownA7: "color(display-p3 0.5569 0.2863 0.0039 / 0.361)",
    brownA8: "color(display-p3 0.549 0.2745 0.0039 / 0.487)",
    brownA9: "color(display-p3 0.4471 0.2196 0 / 0.632)",
    brownA10: "color(display-p3 0.3882 0.1882 0 / 0.655)",
    brownA11: "color(display-p3 0.485 0.3742 0.2884)",
    brownA12: "color(display-p3 0.2361 0.2016 0.1832)",
};
const bronze = {
    bronze1: "#fdfcfc",
    bronze2: "#fdf7f5",
    bronze3: "#f6edea",
    bronze4: "#efe4df",
    bronze5: "#e7d9d3",
    bronze6: "#dfcdc5",
    bronze7: "#d3bcb3",
    bronze8: "#c2a499",
    bronze9: "#a18072",
    bronze10: "#957468",
    bronze11: "#7d5e54",
    bronze12: "#43302b",
};
const bronzeA = {
    bronzeA1: "#55000003",
    bronzeA2: "#cc33000a",
    bronzeA3: "#92250015",
    bronzeA4: "#80280020",
    bronzeA5: "#7423002c",
    bronzeA6: "#7324003a",
    bronzeA7: "#6c1f004c",
    bronzeA8: "#671c0066",
    bronzeA9: "#551a008d",
    bronzeA10: "#4c150097",
    bronzeA11: "#3d0f00ab",
    bronzeA12: "#1d0600d4",
};
const bronzeP3 = {
    bronze1: "color(display-p3 0.9915 0.9884 0.9883)",
    bronze2: "color(display-p3 0.9887 0.9701 0.9605)",
    bronze3: "color(display-p3 0.9584 0.9321 0.9189)",
    bronze4: "color(display-p3 0.9288 0.8942 0.877)",
    bronze5: "color(display-p3 0.8977 0.8534 0.8319)",
    bronze6: "color(display-p3 0.8611 0.8045 0.7778)",
    bronze7: "color(display-p3 0.8118 0.7392 0.706)",
    bronze8: "color(display-p3 0.7409 0.647 0.6064)",
    bronze9: "color(display-p3 0.6109 0.5069 0.4552)",
    bronze10: "color(display-p3 0.5629 0.4607 0.414)",
    bronze11: "color(display-p3 0.4714 0.3735 0.336)",
    bronze12: "color(display-p3 0.2513 0.1912 0.1722)",
};
const bronzeP3A = {
    bronzeA1: "color(display-p3 0.349 0.0235 0.0235 / 0.012)",
    bronzeA2: "color(display-p3 0.7098 0.2196 0.0235 / 0.04)",
    bronzeA3: "color(display-p3 0.4824 0.2 0.0078 / 0.083)",
    bronzeA4: "color(display-p3 0.4235 0.1333 0.0039 / 0.122)",
    bronzeA5: "color(display-p3 0.4 0.1451 0.0039 / 0.169)",
    bronzeA6: "color(display-p3 0.3882 0.1255 0.0039 / 0.224)",
    bronzeA7: "color(display-p3 0.3647 0.1098 0.0039 / 0.295)",
    bronzeA8: "color(display-p3 0.3412 0.102 0.0039 / 0.393)",
    bronzeA9: "color(display-p3 0.2902 0.0941 0 / 0.546)",
    bronzeA10: "color(display-p3 0.2549 0.0745 0 / 0.585)",
    bronzeA11: "color(display-p3 0.4714 0.3735 0.336)",
    bronzeA12: "color(display-p3 0.2513 0.1912 0.1722)",
};
const gold = {
    gold1: "#fdfdfc",
    gold2: "#faf9f2",
    gold3: "#f2f0e7",
    gold4: "#eae6db",
    gold5: "#e1dccf",
    gold6: "#d8d0bf",
    gold7: "#cbc0aa",
    gold8: "#b9a88d",
    gold9: "#978365",
    gold10: "#8c7a5e",
    gold11: "#71624b",
    gold12: "#3b352b",
};
const goldA = {
    goldA1: "#55550003",
    goldA2: "#9d8a000d",
    goldA3: "#75600018",
    goldA4: "#6b4e0024",
    goldA5: "#60460030",
    goldA6: "#64440040",
    goldA7: "#63420055",
    goldA8: "#633d0072",
    goldA9: "#5332009a",
    goldA10: "#492d00a1",
    goldA11: "#362100b4",
    goldA12: "#130c00d4",
};
const goldP3 = {
    gold1: "color(display-p3 0.9922 0.9922 0.9886)",
    gold2: "color(display-p3 0.9804 0.9764 0.9531)",
    gold3: "color(display-p3 0.9467 0.9401 0.9093)",
    gold4: "color(display-p3 0.9141 0.9039 0.8647)",
    gold5: "color(display-p3 0.8801 0.8648 0.8157)",
    gold6: "color(display-p3 0.8404 0.8177 0.7562)",
    gold7: "color(display-p3 0.7883 0.7535 0.6766)",
    gold8: "color(display-p3 0.7148 0.6597 0.5654)",
    gold9: "color(display-p3 0.5792 0.5166 0.4103)",
    gold10: "color(display-p3 0.5376 0.4795 0.3803)",
    gold11: "color(display-p3 0.4334 0.3864 0.305)",
    gold12: "color(display-p3 0.2274 0.2087 0.1731)",
};
const goldP3A = {
    goldA1: "color(display-p3 0.349 0.349 0.0235 / 0.012)",
    goldA2: "color(display-p3 0.5922 0.5137 0.0235 / 0.048)",
    goldA3: "color(display-p3 0.4 0.3569 0.0118 / 0.091)",
    goldA4: "color(display-p3 0.3765 0.2902 0.0078 / 0.138)",
    goldA5: "color(display-p3 0.3451 0.2824 0.0039 / 0.185)",
    goldA6: "color(display-p3 0.3412 0.2627 0.0039 / 0.244)",
    goldA7: "color(display-p3 0.3451 0.2353 0.0039 / 0.322)",
    goldA8: "color(display-p3 0.3451 0.2196 0.0039 / 0.436)",
    goldA9: "color(display-p3 0.2863 0.1804 0 / 0.589)",
    goldA10: "color(display-p3 0.2549 0.1608 0 / 0.62)",
    goldA11: "color(display-p3 0.4334 0.3864 0.305)",
    goldA12: "color(display-p3 0.2274 0.2087 0.1731)",
};
const sky = {
    sky1: "#f9feff",
    sky2: "#eefbff",
    sky3: "#ddf7fe",
    sky4: "#c5f2ff",
    sky5: "#b3e9fb",
    sky6: "#a0dbf0",
    sky7: "#86c8e3",
    sky8: "#60b3d7",
    sky9: "#7ce2fe",
    sky10: "#74daf8",
    sky11: "#00749e",
    sky12: "#1d3e56",
};
const skyA = {
    skyA1: "#00d5ff06",
    skyA2: "#00c3ff11",
    skyA3: "#00c3f822",
    skyA4: "#00c6ff3a",
    skyA5: "#00b6f24c",
    skyA6: "#009fd75f",
    skyA7: "#008cc479",
    skyA8: "#0085bf9f",
    skyA9: "#00c7fe83",
    skyA10: "#00bcf38b",
    skyA11: "#00749e",
    skyA12: "#002540e2",
};
const skyP3 = {
    sky1: "color(display-p3 0.98 0.9954 0.9993)",
    sky2: "color(display-p3 0.9435 0.984 0.9979)",
    sky3: "color(display-p3 0.8846 0.9658 0.9912)",
    sky4: "color(display-p3 0.8088 0.9457 0.9979)",
    sky5: "color(display-p3 0.7449 0.908 0.975)",
    sky6: "color(display-p3 0.6749 0.8523 0.9327)",
    sky7: "color(display-p3 0.5832 0.7788 0.8783)",
    sky8: "color(display-p3 0.457 0.6958 0.8295)",
    sky9: "color(display-p3 0.5845 0.8766 0.9826)",
    sky10: "color(display-p3 0.5546 0.8454 0.9586)",
    sky11: "color(display-p3 0.193 0.4475 0.605)",
    sky12: "color(display-p3 0.1452 0.2406 0.3293)",
};
const skyP3A = {
    skyA1: "color(display-p3 0.0196 0.8039 1 / 0.02)",
    skyA2: "color(display-p3 0.0039 0.7176 0.9294 / 0.055)",
    skyA3: "color(display-p3 0.0039 0.6941 0.9333 / 0.114)",
    skyA4: "color(display-p3 0.0078 0.7176 0.9804 / 0.193)",
    skyA5: "color(display-p3 0.0039 0.6471 0.9098 / 0.255)",
    skyA6: "color(display-p3 0.0039 0.5451 0.7961 / 0.326)",
    skyA7: "color(display-p3 0.0039 0.4745 0.7098 / 0.416)",
    skyA8: "color(display-p3 0 0.4353 0.6902 / 0.542)",
    skyA9: "color(display-p3 0.0039 0.7098 0.9647 / 0.416)",
    skyA10: "color(display-p3 0.0039 0.6588 0.9059 / 0.448)",
    skyA11: "color(display-p3 0.193 0.4475 0.605)",
    skyA12: "color(display-p3 0.1452 0.2406 0.3293)",
};
const mint = {
    mint1: "#f9fefd",
    mint2: "#eefefa",
    mint3: "#dbfaf3",
    mint4: "#c4f6eb",
    mint5: "#b2eee1",
    mint6: "#9ee1d2",
    mint7: "#80d0bf",
    mint8: "#51bda8",
    mint9: "#86ead4",
    mint10: "#7de0cb",
    mint11: "#027864",
    mint12: "#16433c",
};
const mintA = {
    mintA1: "#00d5aa06",
    mintA2: "#00f0b411",
    mintA3: "#00dcaa24",
    mintA4: "#01d9a93b",
    mintA5: "#00c79c4d",
    mintA6: "#00b18961",
    mintA7: "#00a17f7f",
    mintA8: "#009f80ae",
    mintA9: "#00d3a579",
    mintA10: "#00c39982",
    mintA11: "#007763fd",
    mintA12: "#00312ae9",
};
const mintP3 = {
    mint1: "color(display-p3 0.98 0.9954 0.9922)",
    mint2: "color(display-p3 0.944 0.9937 0.9812)",
    mint3: "color(display-p3 0.8829 0.9763 0.9544)",
    mint4: "color(display-p3 0.8088 0.9589 0.9221)",
    mint5: "color(display-p3 0.7484 0.9261 0.8825)",
    mint6: "color(display-p3 0.6766 0.8759 0.8265)",
    mint7: "color(display-p3 0.5743 0.8087 0.7503)",
    mint8: "color(display-p3 0.4347 0.7322 0.6595)",
    mint9: "color(display-p3 0.6198 0.908 0.834)",
    mint10: "color(display-p3 0.5851 0.8706 0.797)",
    mint11: "color(display-p3 0.2026 0.4631 0.3967)",
    mint12: "color(display-p3 0.1361 0.2589 0.2357)",
};
const mintP3A = {
    mintA1: "color(display-p3 0.0196 0.8039 0.6078 / 0.02)",
    mintA2: "color(display-p3 0.0039 0.8588 0.6471 / 0.055)",
    mintA3: "color(display-p3 0.0039 0.8039 0.6039 / 0.118)",
    mintA4: "color(display-p3 0.0078 0.8 0.5961 / 0.193)",
    mintA5: "color(display-p3 0.0039 0.7059 0.5333 / 0.251)",
    mintA6: "color(display-p3 0.0039 0.6118 0.4667 / 0.322)",
    mintA7: "color(display-p3 0.0039 0.5529 0.4157 / 0.428)",
    mintA8: "color(display-p3 0 0.5294 0.3961 / 0.565)",
    mintA9: "color(display-p3 0.0039 0.7647 0.5686 / 0.381)",
    mintA10: "color(display-p3 0.0039 0.6902 0.5098 / 0.416)",
    mintA11: "color(display-p3 0.2026 0.4631 0.3967)",
    mintA12: "color(display-p3 0.1361 0.2589 0.2357)",
};
const lime = {
    lime1: "#fcfdfa",
    lime2: "#f7faed",
    lime3: "#ecf6d2",
    lime4: "#e1f2b7",
    lime5: "#d3eaa1",
    lime6: "#c2dc8d",
    lime7: "#abca77",
    lime8: "#8db654",
    lime9: "#bdee63",
    lime10: "#b0e64c",
    lime11: "#5c7c2f",
    lime12: "#37401c",
};
const limeA = {
    limeA1: "#66990005",
    limeA2: "#8eb90012",
    limeA3: "#94cc002d",
    limeA4: "#95d10048",
    limeA5: "#88c7005e",
    limeA6: "#77b10072",
    limeA7: "#629c0088",
    limeA8: "#559200ab",
    limeA9: "#93e4009c",
    limeA10: "#8fdc00b3",
    limeA11: "#375f00d0",
    limeA12: "#1e2900e3",
};
const limeP3 = {
    lime1: "color(display-p3 0.9889 0.992 0.9814)",
    lime2: "color(display-p3 0.9709 0.9816 0.9328)",
    lime3: "color(display-p3 0.9327 0.9641 0.8377)",
    lime4: "color(display-p3 0.8948 0.9479 0.7412)",
    lime5: "color(display-p3 0.8436 0.9135 0.6612)",
    lime6: "color(display-p3 0.779 0.8613 0.5874)",
    lime7: "color(display-p3 0.694 0.7886 0.5029)",
    lime8: "color(display-p3 0.5846 0.7074 0.3784)",
    lime9: "color(display-p3 0.7799 0.9278 0.4663)",
    lime10: "color(display-p3 0.7337 0.896 0.3966)",
    lime11: "color(display-p3 0.3862 0.4822 0.2273)",
    lime12: "color(display-p3 0.2224 0.2499 0.128)",
};
const limeP3A = {
    limeA1: "color(display-p3 0.4118 0.6078 0.0196 / 0.02)",
    limeA2: "color(display-p3 0.5922 0.7098 0.0078 / 0.067)",
    limeA3: "color(display-p3 0.5882 0.7843 0.0039 / 0.161)",
    limeA4: "color(display-p3 0.5922 0.8039 0.0039 / 0.259)",
    limeA5: "color(display-p3 0.5373 0.7451 0.0039 / 0.338)",
    limeA6: "color(display-p3 0.4706 0.6706 0.0039 / 0.412)",
    limeA7: "color(display-p3 0.3882 0.5765 0.0039 / 0.499)",
    limeA8: "color(display-p3 0.3333 0.5294 0 / 0.624)",
    limeA9: "color(display-p3 0.5882 0.8667 0 / 0.534)",
    limeA10: "color(display-p3 0.5608 0.8275 0 / 0.604)",
    limeA11: "color(display-p3 0.3862 0.4822 0.2273)",
    limeA12: "color(display-p3 0.2224 0.2499 0.128)",
};
const yellow = {
    yellow1: "#fdfdf9",
    yellow2: "#fefbe7",
    yellow3: "#fdf7c1",
    yellow4: "#fcef97",
    yellow5: "#ffe770",
    yellow6: "#f0d775",
    yellow7: "#e7cf70",
    yellow8: "#dab74e",
    yellow9: "#ffe629",
    yellow10: "#ffdc00",
    yellow11: "#9e6c00",
    yellow12: "#473b1f",
};
const yellowA = {
    yellowA1: "#aaaa0006",
    yellowA2: "#f5d50018",
    yellowA3: "#f7df003e",
    yellowA4: "#f8d80068",
    yellowA5: "#ffd5008f",
    yellowA6: "#e4b6008a",
    yellowA7: "#d5aa008f",
    yellowA8: "#ca9700b1",
    yellowA9: "#ffe100d6",
    yellowA10: "#ffdc00",
    yellowA11: "#9e6c00",
    yellowA12: "#2e2000e0",
};
const yellowP3 = {
    yellow1: "color(display-p3 0.9922 0.9922 0.9779)",
    yellow2: "color(display-p3 0.9931 0.9849 0.9127)",
    yellow3: "color(display-p3 0.9884 0.9698 0.7801)",
    yellow4: "color(display-p3 0.9803 0.9381 0.6335)",
    yellow5: "color(display-p3 0.9843 0.91 0.5097)",
    yellow6: "color(display-p3 0.9239 0.8474 0.5138)",
    yellow7: "color(display-p3 0.8904 0.8154 0.4907)",
    yellow8: "color(display-p3 0.8309 0.7216 0.3749)",
    yellow9: "color(display-p3 1 0.92 0.22)",
    yellow10: "color(display-p3 0.9775 0.8678 0.2911)",
    yellow11: "color(display-p3 0.6 0.44 0)",
    yellow12: "color(display-p3 0.2708 0.2331 0.1368)",
};
const yellowP3A = {
    yellowA1: "color(display-p3 0.6745 0.6745 0.0235 / 0.024)",
    yellowA2: "color(display-p3 0.9137 0.8235 0.0118 / 0.087)",
    yellowA3: "color(display-p3 0.949 0.8588 0.0039 / 0.22)",
    yellowA4: "color(display-p3 0.949 0.8314 0.0039 / 0.365)",
    yellowA5: "color(display-p3 0.9686 0.8157 0.0039 / 0.491)",
    yellowA6: "color(display-p3 0.8471 0.6863 0.0039 / 0.487)",
    yellowA7: "color(display-p3 0.7843 0.6392 0 / 0.51)",
    yellowA8: "color(display-p3 0.7294 0.5529 0 / 0.624)",
    yellowA9: "color(display-p3 1 0.898 0 / 0.781)",
    yellowA10: "color(display-p3 0.9686 0.8118 0 / 0.71)",
    yellowA11: "color(display-p3 0.6 0.44 0)",
    yellowA12: "color(display-p3 0.2708 0.2331 0.1368)",
};
const amber = {
    amber1: "#fefdfb",
    amber2: "#fff9e8",
    amber3: "#fff4c7",
    amber4: "#ffeb9c",
    amber5: "#ffe176",
    amber6: "#fad373",
    amber7: "#edbf63",
    amber8: "#e2a336",
    amber9: "#ffc53d",
    amber10: "#ffba18",
    amber11: "#ab6400",
    amber12: "#4f3422",
};
const amberA = {
    amberA1: "#c0800004",
    amberA2: "#ffbd0017",
    amberA3: "#ffcd0038",
    amberA4: "#ffcc0063",
    amberA5: "#ffc80089",
    amberA6: "#f6af008c",
    amberA7: "#e297009c",
    amberA8: "#da8a00c9",
    amberA9: "#ffb300c2",
    amberA10: "#ffb300e7",
    amberA11: "#ab6400",
    amberA12: "#341500dd",
};
const amberP3 = {
    amber1: "color(display-p3 0.9954 0.9923 0.9851)",
    amber2: "color(display-p3 0.9961 0.9783 0.9167)",
    amber3: "color(display-p3 0.9924 0.9575 0.7988)",
    amber4: "color(display-p3 0.9867 0.9247 0.6485)",
    amber5: "color(display-p3 0.9969 0.8863 0.5256)",
    amber6: "color(display-p3 0.9539 0.8325 0.5064)",
    amber7: "color(display-p3 0.8993 0.7553 0.4444)",
    amber8: "color(display-p3 0.8503 0.6503 0.3004)",
    amber9: "color(display-p3 1 0.77 0.26)",
    amber10: "color(display-p3 0.9592 0.7405 0.2738)",
    amber11: "color(display-p3 0.64 0.4 0)",
    amber12: "color(display-p3 0.2942 0.2084 0.1446)",
};
const amberP3A = {
    amberA1: "color(display-p3 0.7569 0.5137 0.0235 / 0.016)",
    amberA2: "color(display-p3 0.9529 0.7176 0.0078 / 0.083)",
    amberA3: "color(display-p3 0.9608 0.7843 0 / 0.2)",
    amberA4: "color(display-p3 0.9686 0.7922 0.0039 / 0.353)",
    amberA5: "color(display-p3 0.9922 0.7608 0.0039 / 0.475)",
    amberA6: "color(display-p3 0.9059 0.6588 0.0039 / 0.495)",
    amberA7: "color(display-p3 0.8196 0.5647 0 / 0.557)",
    amberA8: "color(display-p3 0.7882 0.502 0 / 0.699)",
    amberA9: "color(display-p3 1 0.6863 0 / 0.742)",
    amberA10: "color(display-p3 0.9451 0.6431 0 / 0.726)",
    amberA11: "color(display-p3 0.64 0.4 0)",
    amberA12: "color(display-p3 0.2942 0.2084 0.1446)",
};
const orange = {
    orange1: "#fefcfb",
    orange2: "#fff8f0",
    orange3: "#ffefd6",
    orange4: "#ffe0b7",
    orange5: "#ffd29c",
    orange6: "#ffc182",
    orange7: "#f5ae73",
    orange8: "#ec9455",
    orange9: "#f76b15",
    orange10: "#ef5f00",
    orange11: "#cc4e00",
    orange12: "#582d1d",
};
const orangeA = {
    orangeA1: "#c0400004",
    orangeA2: "#ff88000f",
    orangeA3: "#ff9c0029",
    orangeA4: "#ff920048",
    orangeA5: "#ff8c0063",
    orangeA6: "#ff81007d",
    orangeA7: "#ed6c008c",
    orangeA8: "#e35f00aa",
    orangeA9: "#f65e00ea",
    orangeA10: "#ef5f00",
    orangeA11: "#cc4e00",
    orangeA12: "#431200e2",
};
const orangeP3 = {
    orange1: "color(display-p3 0.9947 0.9885 0.9848)",
    orange2: "color(display-p3 0.9951 0.9729 0.9434)",
    orange3: "color(display-p3 0.989 0.9382 0.8505)",
    orange4: "color(display-p3 1 0.8775 0.6958)",
    orange5: "color(display-p3 1 0.8237 0.5864)",
    orange6: "color(display-p3 0.9766 0.7689 0.548)",
    orange7: "color(display-p3 0.9192 0.6946 0.4879)",
    orange8: "color(display-p3 0.8774 0.5965 0.3794)",
    orange9: "color(display-p3 0.9 0.45 0.2)",
    orange10: "color(display-p3 0.8705 0.409 0.1642)",
    orange11: "color(display-p3 0.76 0.34 0)",
    orange12: "color(display-p3 0.3226 0.185 0.1271)",
};
const orangeP3A = {
    orangeA1: "color(display-p3 0.7569 0.2667 0.0235 / 0.016)",
    orangeA2: "color(display-p3 0.9294 0.502 0.0039 / 0.055)",
    orangeA3: "color(display-p3 0.9216 0.5843 0.0078 / 0.15)",
    orangeA4: "color(display-p3 1 0.6039 0.0039 / 0.306)",
    orangeA5: "color(display-p3 1 0.5725 0.0039 / 0.412)",
    orangeA6: "color(display-p3 0.949 0.4902 0.0039 / 0.451)",
    orangeA7: "color(display-p3 0.8392 0.4078 0 / 0.514)",
    orangeA8: "color(display-p3 0.8039 0.349 0 / 0.62)",
    orangeA9: "color(display-p3 0.8784 0.3137 0 / 0.8)",
    orangeA10: "color(display-p3 0.8431 0.2902 0 / 0.836)",
    orangeA11: "color(display-p3 0.76 0.34 0)",
    orangeA12: "color(display-p3 0.3226 0.185 0.1271)",
};

const blackA = {
    blackA1: "rgba(0, 0, 0, 0.05)",
    blackA2: "rgba(0, 0, 0, 0.1)",
    blackA3: "rgba(0, 0, 0, 0.15)",
    blackA4: "rgba(0, 0, 0, 0.2)",
    blackA5: "rgba(0, 0, 0, 0.3)",
    blackA6: "rgba(0, 0, 0, 0.4)",
    blackA7: "rgba(0, 0, 0, 0.5)",
    blackA8: "rgba(0, 0, 0, 0.6)",
    blackA9: "rgba(0, 0, 0, 0.7)",
    blackA10: "rgba(0, 0, 0, 0.8)",
    blackA11: "rgba(0, 0, 0, 0.9)",
    blackA12: "rgba(0, 0, 0, 0.95)",
};
const blackP3A = {
    blackA1: "color(display-p3 0 0 0 / 0.05)",
    blackA2: "color(display-p3 0 0 0 / 0.1)",
    blackA3: "color(display-p3 0 0 0 / 0.15)",
    blackA4: "color(display-p3 0 0 0 / 0.2)",
    blackA5: "color(display-p3 0 0 0 / 0.3)",
    blackA6: "color(display-p3 0 0 0 / 0.4)",
    blackA7: "color(display-p3 0 0 0 / 0.5)",
    blackA8: "color(display-p3 0 0 0 / 0.6)",
    blackA9: "color(display-p3 0 0 0 / 0.7)",
    blackA10: "color(display-p3 0 0 0 / 0.8)",
    blackA11: "color(display-p3 0 0 0 / 0.9)",
    blackA12: "color(display-p3 0 0 0 / 0.95)",
};

const whiteA = {
    whiteA1: "rgba(255, 255, 255, 0.05)",
    whiteA2: "rgba(255, 255, 255, 0.1)",
    whiteA3: "rgba(255, 255, 255, 0.15)",
    whiteA4: "rgba(255, 255, 255, 0.2)",
    whiteA5: "rgba(255, 255, 255, 0.3)",
    whiteA6: "rgba(255, 255, 255, 0.4)",
    whiteA7: "rgba(255, 255, 255, 0.5)",
    whiteA8: "rgba(255, 255, 255, 0.6)",
    whiteA9: "rgba(255, 255, 255, 0.7)",
    whiteA10: "rgba(255, 255, 255, 0.8)",
    whiteA11: "rgba(255, 255, 255, 0.9)",
    whiteA12: "rgba(255, 255, 255, 0.95)",
};
const whiteP3A = {
    whiteA1: "color(display-p3 1 1 1 / 0.05)",
    whiteA2: "color(display-p3 1 1 1 / 0.1)",
    whiteA3: "color(display-p3 1 1 1 / 0.15)",
    whiteA4: "color(display-p3 1 1 1 / 0.2)",
    whiteA5: "color(display-p3 1 1 1 / 0.3)",
    whiteA6: "color(display-p3 1 1 1 / 0.4)",
    whiteA7: "color(display-p3 1 1 1 / 0.5)",
    whiteA8: "color(display-p3 1 1 1 / 0.6)",
    whiteA9: "color(display-p3 1 1 1 / 0.7)",
    whiteA10: "color(display-p3 1 1 1 / 0.8)",
    whiteA11: "color(display-p3 1 1 1 / 0.9)",
    whiteA12: "color(display-p3 1 1 1 / 0.95)",
};

exports.amber = amber;
exports.amberA = amberA;
exports.amberDark = amberDark;
exports.amberDarkA = amberDarkA;
exports.amberDarkP3 = amberDarkP3;
exports.amberDarkP3A = amberDarkP3A;
exports.amberP3 = amberP3;
exports.amberP3A = amberP3A;
exports.blackA = blackA;
exports.blackP3A = blackP3A;
exports.blue = blue;
exports.blueA = blueA;
exports.blueDark = blueDark;
exports.blueDarkA = blueDarkA;
exports.blueDarkP3 = blueDarkP3;
exports.blueDarkP3A = blueDarkP3A;
exports.blueP3 = blueP3;
exports.blueP3A = blueP3A;
exports.bronze = bronze;
exports.bronzeA = bronzeA;
exports.bronzeDark = bronzeDark;
exports.bronzeDarkA = bronzeDarkA;
exports.bronzeDarkP3 = bronzeDarkP3;
exports.bronzeDarkP3A = bronzeDarkP3A;
exports.bronzeP3 = bronzeP3;
exports.bronzeP3A = bronzeP3A;
exports.brown = brown;
exports.brownA = brownA;
exports.brownDark = brownDark;
exports.brownDarkA = brownDarkA;
exports.brownDarkP3 = brownDarkP3;
exports.brownDarkP3A = brownDarkP3A;
exports.brownP3 = brownP3;
exports.brownP3A = brownP3A;
exports.crimson = crimson;
exports.crimsonA = crimsonA;
exports.crimsonDark = crimsonDark;
exports.crimsonDarkA = crimsonDarkA;
exports.crimsonDarkP3 = crimsonDarkP3;
exports.crimsonDarkP3A = crimsonDarkP3A;
exports.crimsonP3 = crimsonP3;
exports.crimsonP3A = crimsonP3A;
exports.cyan = cyan;
exports.cyanA = cyanA;
exports.cyanDark = cyanDark;
exports.cyanDarkA = cyanDarkA;
exports.cyanDarkP3 = cyanDarkP3;
exports.cyanDarkP3A = cyanDarkP3A;
exports.cyanP3 = cyanP3;
exports.cyanP3A = cyanP3A;
exports.gold = gold;
exports.goldA = goldA;
exports.goldDark = goldDark;
exports.goldDarkA = goldDarkA;
exports.goldDarkP3 = goldDarkP3;
exports.goldDarkP3A = goldDarkP3A;
exports.goldP3 = goldP3;
exports.goldP3A = goldP3A;
exports.grass = grass;
exports.grassA = grassA;
exports.grassDark = grassDark;
exports.grassDarkA = grassDarkA;
exports.grassDarkP3 = grassDarkP3;
exports.grassDarkP3A = grassDarkP3A;
exports.grassP3 = grassP3;
exports.grassP3A = grassP3A;
exports.gray = gray;
exports.grayA = grayA;
exports.grayDark = grayDark;
exports.grayDarkA = grayDarkA;
exports.grayDarkP3 = grayDarkP3;
exports.grayDarkP3A = grayDarkP3A;
exports.grayP3 = grayP3;
exports.grayP3A = grayP3A;
exports.green = green;
exports.greenA = greenA;
exports.greenDark = greenDark;
exports.greenDarkA = greenDarkA;
exports.greenDarkP3 = greenDarkP3;
exports.greenDarkP3A = greenDarkP3A;
exports.greenP3 = greenP3;
exports.greenP3A = greenP3A;
exports.indigo = indigo;
exports.indigoA = indigoA;
exports.indigoDark = indigoDark;
exports.indigoDarkA = indigoDarkA;
exports.indigoDarkP3 = indigoDarkP3;
exports.indigoDarkP3A = indigoDarkP3A;
exports.indigoP3 = indigoP3;
exports.indigoP3A = indigoP3A;
exports.iris = iris;
exports.irisA = irisA;
exports.irisDark = irisDark;
exports.irisDarkA = irisDarkA;
exports.irisDarkP3 = irisDarkP3;
exports.irisDarkP3A = irisDarkP3A;
exports.irisP3 = irisP3;
exports.irisP3A = irisP3A;
exports.jade = jade;
exports.jadeA = jadeA;
exports.jadeDark = jadeDark;
exports.jadeDarkA = jadeDarkA;
exports.jadeDarkP3 = jadeDarkP3;
exports.jadeDarkP3A = jadeDarkP3A;
exports.jadeP3 = jadeP3;
exports.jadeP3A = jadeP3A;
exports.lime = lime;
exports.limeA = limeA;
exports.limeDark = limeDark;
exports.limeDarkA = limeDarkA;
exports.limeDarkP3 = limeDarkP3;
exports.limeDarkP3A = limeDarkP3A;
exports.limeP3 = limeP3;
exports.limeP3A = limeP3A;
exports.mauve = mauve;
exports.mauveA = mauveA;
exports.mauveDark = mauveDark;
exports.mauveDarkA = mauveDarkA;
exports.mauveDarkP3 = mauveDarkP3;
exports.mauveDarkP3A = mauveDarkP3A;
exports.mauveP3 = mauveP3;
exports.mauveP3A = mauveP3A;
exports.mint = mint;
exports.mintA = mintA;
exports.mintDark = mintDark;
exports.mintDarkA = mintDarkA;
exports.mintDarkP3 = mintDarkP3;
exports.mintDarkP3A = mintDarkP3A;
exports.mintP3 = mintP3;
exports.mintP3A = mintP3A;
exports.olive = olive;
exports.oliveA = oliveA;
exports.oliveDark = oliveDark;
exports.oliveDarkA = oliveDarkA;
exports.oliveDarkP3 = oliveDarkP3;
exports.oliveDarkP3A = oliveDarkP3A;
exports.oliveP3 = oliveP3;
exports.oliveP3A = oliveP3A;
exports.orange = orange;
exports.orangeA = orangeA;
exports.orangeDark = orangeDark;
exports.orangeDarkA = orangeDarkA;
exports.orangeDarkP3 = orangeDarkP3;
exports.orangeDarkP3A = orangeDarkP3A;
exports.orangeP3 = orangeP3;
exports.orangeP3A = orangeP3A;
exports.pink = pink;
exports.pinkA = pinkA;
exports.pinkDark = pinkDark;
exports.pinkDarkA = pinkDarkA;
exports.pinkDarkP3 = pinkDarkP3;
exports.pinkDarkP3A = pinkDarkP3A;
exports.pinkP3 = pinkP3;
exports.pinkP3A = pinkP3A;
exports.plum = plum;
exports.plumA = plumA;
exports.plumDark = plumDark;
exports.plumDarkA = plumDarkA;
exports.plumDarkP3 = plumDarkP3;
exports.plumDarkP3A = plumDarkP3A;
exports.plumP3 = plumP3;
exports.plumP3A = plumP3A;
exports.purple = purple;
exports.purpleA = purpleA;
exports.purpleDark = purpleDark;
exports.purpleDarkA = purpleDarkA;
exports.purpleDarkP3 = purpleDarkP3;
exports.purpleDarkP3A = purpleDarkP3A;
exports.purpleP3 = purpleP3;
exports.purpleP3A = purpleP3A;
exports.red = red;
exports.redA = redA;
exports.redDark = redDark;
exports.redDarkA = redDarkA;
exports.redDarkP3 = redDarkP3;
exports.redDarkP3A = redDarkP3A;
exports.redP3 = redP3;
exports.redP3A = redP3A;
exports.ruby = ruby;
exports.rubyA = rubyA;
exports.rubyDark = rubyDark;
exports.rubyDarkA = rubyDarkA;
exports.rubyDarkP3 = rubyDarkP3;
exports.rubyDarkP3A = rubyDarkP3A;
exports.rubyP3 = rubyP3;
exports.rubyP3A = rubyP3A;
exports.sage = sage;
exports.sageA = sageA;
exports.sageDark = sageDark;
exports.sageDarkA = sageDarkA;
exports.sageDarkP3 = sageDarkP3;
exports.sageDarkP3A = sageDarkP3A;
exports.sageP3 = sageP3;
exports.sageP3A = sageP3A;
exports.sand = sand;
exports.sandA = sandA;
exports.sandDark = sandDark;
exports.sandDarkA = sandDarkA;
exports.sandDarkP3 = sandDarkP3;
exports.sandDarkP3A = sandDarkP3A;
exports.sandP3 = sandP3;
exports.sandP3A = sandP3A;
exports.sky = sky;
exports.skyA = skyA;
exports.skyDark = skyDark;
exports.skyDarkA = skyDarkA;
exports.skyDarkP3 = skyDarkP3;
exports.skyDarkP3A = skyDarkP3A;
exports.skyP3 = skyP3;
exports.skyP3A = skyP3A;
exports.slate = slate;
exports.slateA = slateA;
exports.slateDark = slateDark;
exports.slateDarkA = slateDarkA;
exports.slateDarkP3 = slateDarkP3;
exports.slateDarkP3A = slateDarkP3A;
exports.slateP3 = slateP3;
exports.slateP3A = slateP3A;
exports.teal = teal;
exports.tealA = tealA;
exports.tealDark = tealDark;
exports.tealDarkA = tealDarkA;
exports.tealDarkP3 = tealDarkP3;
exports.tealDarkP3A = tealDarkP3A;
exports.tealP3 = tealP3;
exports.tealP3A = tealP3A;
exports.tomato = tomato;
exports.tomatoA = tomatoA;
exports.tomatoDark = tomatoDark;
exports.tomatoDarkA = tomatoDarkA;
exports.tomatoDarkP3 = tomatoDarkP3;
exports.tomatoDarkP3A = tomatoDarkP3A;
exports.tomatoP3 = tomatoP3;
exports.tomatoP3A = tomatoP3A;
exports.violet = violet;
exports.violetA = violetA;
exports.violetDark = violetDark;
exports.violetDarkA = violetDarkA;
exports.violetDarkP3 = violetDarkP3;
exports.violetDarkP3A = violetDarkP3A;
exports.violetP3 = violetP3;
exports.violetP3A = violetP3A;
exports.whiteA = whiteA;
exports.whiteP3A = whiteP3A;
exports.yellow = yellow;
exports.yellowA = yellowA;
exports.yellowDark = yellowDark;
exports.yellowDarkA = yellowDarkA;
exports.yellowDarkP3 = yellowDarkP3;
exports.yellowDarkP3A = yellowDarkP3A;
exports.yellowP3 = yellowP3;
exports.yellowP3A = yellowP3A;
